<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />


<meta name="author" content="Salomé Fromonteil1, Lennart Winkler2, Lucas Marie-Orleach3,4, Tim Janicke1,2*   1Centre d’Écologie Fonctionnelle et Évolutive, UMR 5175, CNRS, Université de Montpellier   2Applied Zoology, Technical University Dresden   3Natural History Museum, University of Oslo   4ECOBIO (Écosystèmes, biodiversité, évolution), UMR 6553, CNRS, Université de Rennes 1   *Corresponding author: tim.janicke@cefe.cnrs.fr" />


<title>Sexual selection in females across the animal tree of life</title>

<script src="site_libs/header-attrs-2.11/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/paper.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/jqueryui-1.11.4/jquery-ui.min.js"></script>
<link href="site_libs/tocify-1.9.1/jquery.tocify.css" rel="stylesheet" />
<script src="site_libs/tocify-1.9.1/jquery.tocify.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<script src="site_libs/navigation-1.1/codefolding.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<script src="site_libs/kePrint-0.0.1/kePrint.js"></script>
<link href="site_libs/lightable-0.0.1/lightable.css" rel="stylesheet" />

<link rel="icon" href="https://github.com/workflowr/workflowr-assets/raw/master/img/reproducible.png">
<!-- Add a small amount of space between sections. -->
<style type="text/css">
div.section {
  padding-top: 12px;
}
</style>



<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>




<style type="text/css">
/* for pandoc --citeproc since 2.11 */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>




<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->
<style type="text/css">
.code-folding-btn { margin-bottom: 4px; }
</style>



<style type="text/css">

#TOC {
  margin: 25px 0px 20px 0px;
}
@media (max-width: 768px) {
#TOC {
  position: relative;
  width: 100%;
}
}

@media print {
.toc-content {
  /* see https://github.com/w3c/csswg-drafts/issues/4434 */
  float: right;
}
}

.toc-content {
  padding-left: 30px;
  padding-right: 40px;
}

div.main-container {
  max-width: 1200px;
}

div.tocify {
  width: 20%;
  max-width: 260px;
  max-height: 85%;
}

@media (min-width: 768px) and (max-width: 991px) {
  div.tocify {
    width: 25%;
  }
}

@media (max-width: 767px) {
  div.tocify {
    width: 100%;
    max-width: none;
  }
}

.tocify ul, .tocify li {
  line-height: 20px;
}

.tocify-subheader .tocify-item {
  font-size: 0.90em;
}

.tocify .list-group-item {
  border-radius: 0px;
}


</style>



</head>

<body>


<div class="container-fluid main-container">


<!-- setup 3col/9col grid for toc_float and main content  -->
<div class="row">
<div class="col-xs-12 col-sm-4 col-md-3">
<div id="TOC" class="tocify">
</div>
</div>

<div class="toc-content col-xs-12 col-sm-8 col-md-9">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">META_SexSelFem</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">Home</a>
</li>
<li>
  <a href="about.html">About</a>
</li>
<li>
  <a href="license.html">License</a>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">

<div class="btn-group pull-right float-right">
<button type="button" class="btn btn-default btn-xs btn-secondary btn-sm dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><span>Code</span> <span class="caret"></span></button>
<ul class="dropdown-menu dropdown-menu-right" style="min-width: 50px;">
<li><a id="rmd-show-all-code" href="#">Show All Code</a></li>
<li><a id="rmd-hide-all-code" href="#">Hide All Code</a></li>
</ul>
</div>



<h1 class="title toc-ignore">Sexual selection in females across the animal tree of life</h1>
<h3 class="subtitle">Supplementary material reporting R code</h3>
<h4 class="author">Salomé Fromonteil<sup>1</sup>, Lennart Winkler<sup>2</sup>, Lucas Marie-Orleach<sup>3,4</sup>, Tim Janicke<sup>1,2</sup>* <br></br> <sup>1</sup>Centre d’Écologie Fonctionnelle et Évolutive, UMR 5175, CNRS, Université de Montpellier <br> <sup>2</sup>Applied Zoology, Technical University Dresden <br> <sup>3</sup>Natural History Museum, University of Oslo <br> <sup>4</sup>ECOBIO (Écosystèmes, biodiversité, évolution), UMR 6553, CNRS, Université de Rennes 1 <br> *Corresponding author: <a href="mailto:tim.janicke@cefe.cnrs.fr" class="email">tim.janicke@cefe.cnrs.fr</a></h4>

</div>


<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-report" data-toggle="collapse" data-target="#workflowr-report">
<span class="glyphicon glyphicon-list" aria-hidden="true"></span> workflowr <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span>
</button>
</p>
<div id="workflowr-report" class="collapse">
<ul class="nav nav-tabs">
<li class="active">
<a data-toggle="tab" href="#summary">Summary</a>
</li>
<li>
<a data-toggle="tab" href="#checks"> Checks <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> </a>
</li>
<li>
<a data-toggle="tab" href="#versions">Past versions</a>
</li>
</ul>
<div class="tab-content">
<div id="summary" class="tab-pane fade in active">
<p>
<strong>Last updated:</strong> 2021-10-21
</p>
<p>
<strong>Checks:</strong> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> 7 <span class="glyphicon glyphicon-exclamation-sign text-danger" aria-hidden="true"></span> 0
</p>
<p>
<strong>Knit directory:</strong> <code>META_SexSelFem/</code> <span class="glyphicon glyphicon-question-sign" aria-hidden="true" title="This is the local directory in which the code in this file was executed."> </span>
</p>
<p>
This reproducible <a href="http://rmarkdown.rstudio.com">R Markdown</a> analysis was created with <a
  href="https://github.com/jdblischak/workflowr">workflowr</a> (version 1.6.2). The <em>Checks</em> tab describes the reproducibility checks that were applied when the results were created. The <em>Past versions</em> tab lists the development history.
</p>
<hr>
</div>
<div id="checks" class="tab-pane fade">
<div id="workflowr-checks" class="panel-group">
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRMarkdownfilestronguptodate"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>R Markdown file:</strong> up-to-date </a>
</p>
</div>
<div id="strongRMarkdownfilestronguptodate" class="panel-collapse collapse">
<div class="panel-body">
<p>Great! Since the R Markdown file has been committed to the Git repository, you know the exact version of the code that produced these results.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongEnvironmentstrongempty"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Environment:</strong> empty </a>
</p>
</div>
<div id="strongEnvironmentstrongempty" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! The global environment was empty. Objects defined in the global environment can affect the analysis in your R Markdown file in unknown ways. For reproduciblity it’s best to always run the code in an empty environment.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSeedstrongcodesetseed20211007code"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Seed:</strong> <code>set.seed(20211007)</code> </a>
</p>
</div>
<div id="strongSeedstrongcodesetseed20211007code" class="panel-collapse collapse">
<div class="panel-body">
<p>The command <code>set.seed(20211007)</code> was run prior to running the code in the R Markdown file. Setting a seed ensures that any results that rely on randomness, e.g. subsampling or permutations, are reproducible.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongSessioninformationstrongrecorded"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Session information:</strong> recorded </a>
</p>
</div>
<div id="strongSessioninformationstrongrecorded" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Recording the operating system, R version, and package versions is critical for reproducibility.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongCachestrongnone"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Cache:</strong> none </a>
</p>
</div>
<div id="strongCachestrongnone" class="panel-collapse collapse">
<div class="panel-body">
<p>Nice! There were no cached chunks for this analysis, so you can be confident that you successfully produced the results during this run.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongFilepathsstrongrelative"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>File paths:</strong> relative </a>
</p>
</div>
<div id="strongFilepathsstrongrelative" class="panel-collapse collapse">
<div class="panel-body">
<p>Great job! Using relative paths to the files within your workflowr project makes it easier to run your code on other machines.</p>
</div>
</div>
</div>
<div class="panel panel-default">
<div class="panel-heading">
<p class="panel-title">
<a data-toggle="collapse" data-parent="#workflowr-checks" href="#strongRepositoryversionstrong125daab"> <span class="glyphicon glyphicon-ok text-success" aria-hidden="true"></span> <strong>Repository version:</strong> 125daab </a>
</p>
</div>
<div id="strongRepositoryversionstrong125daab" class="panel-collapse collapse">
<div class="panel-body">
<p>
Great! You are using Git for version control. Tracking code development and connecting the code version to the results is critical for reproducibility.
</p>
<p>
The results in this page were generated with repository version 125daab. See the <em>Past versions</em> tab to see a history of the changes made to the R Markdown and HTML files.
</p>
<p>
Note that you need to be careful to ensure that all relevant files for the analysis have been committed to Git prior to generating the results (you can use <code>wflow_publish</code> or <code>wflow_git_commit</code>). workflowr only checks the R Markdown file, but you know if there are other scripts or data files that it depends on. Below is the status of the Git repository when the results were generated:
</p>
<pre><code>
Ignored files:
    Ignored:    .Rhistory
    Ignored:    .Rproj.user/
    Ignored:    analysis/figure/

</code></pre>
<p>
Note that any generated files, e.g. HTML, png, CSS, etc., are not included in this status report because it is ok for generated content to have uncommitted changes.
</p>
</div>
</div>
</div>
</div>
<hr>
</div>
<div id="versions" class="tab-pane fade">

<p>
These are the previous versions of the repository in which changes were made to the R Markdown (<code>analysis/META_SexSelFem.Rmd</code>) and HTML (<code>docs/META_SexSelFem.html</code>) files. If you’ve configured a remote Git repository (see <code>?wflow_git_remote</code>), click on the hyperlinks in the table below to view the files as they were in that past version.
</p>
<div class="table-responsive">
<table class="table table-condensed table-hover">
<thead>
<tr>
<th>
File
</th>
<th>
Version
</th>
<th>
Author
</th>
<th>
Date
</th>
<th>
Message
</th>
</tr>
</thead>
<tbody>
<tr>
<td>
Rmd
</td>
<td>
125daab
</td>
<td>
salfromo
</td>
<td>
2021-10-21
</td>
<td>
wflow_publish(files = "analysis/META_SexSelFem.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
f483da5
</td>
<td>
salfromo
</td>
<td>
2021-10-21
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
5ae3d06
</td>
<td>
salfromo
</td>
<td>
2021-10-21
</td>
<td>
wflow_publish(files = "analysis/META_SexSelFem.Rmd")
</td>
</tr>
<tr>
<td>
html
</td>
<td>
ff78733
</td>
<td>
salfromo
</td>
<td>
2021-10-16
</td>
<td>
Build site.
</td>
</tr>
<tr>
<td>
Rmd
</td>
<td>
e195e5e
</td>
<td>
salfromo
</td>
<td>
2021-10-16
</td>
<td>
wflow_publish(files = c("data/1Flatworm.pdf", "data/2Snail.pdf",
</td>
</tr>
</tbody>
</table>
</div>
<hr>
</div>
</div>
</div>
<pre class="r"><code>library(Matrix)
library(metafor)
library(ape)
library(coda)
library(MCMCglmm)
library(ggplot2)
library(cowplot)
library(plyr)
library(dplyr)
library(data.table)
library(ggtree)
library(RColorBrewer)
library(grid)
library(ggnewscale)
library(ggtext)
library(kableExtra)
library(htmlTable)
library(knitr)

knitr::opts_chunk$set(warning = FALSE, message = FALSE) # Suppress warnings in the HTML output
knitr::opts_chunk$set(dev = c(&quot;png&quot;, &quot;pdf&quot;)) # saves figures as png and pdf but prints png</code></pre>
<pre class="r"><code># Plot aesthetics ####
      # Colour palettes
Palette &lt;- colorRampPalette(brewer.pal(n = 9, name = &#39;BrBG&#39;))
TealPalette &lt;- colorRampPalette(Palette(20)[10:20])
CoffeePalette &lt;- colorRampPalette(rev(Palette(20)[1:10]))

      # Plot themes
theme_min &lt;- function(...) {            # removes grids and backgrounds in plots
  theme(panel.border = element_blank(), 
        panel.background = element_rect(fill = &quot;transparent&quot;, colour = NA),
        plot.background = element_rect(fill = &quot;transparent&quot;, colour = NA),
        panel.grid.major = element_blank(),
        panel.grid.minor = element_blank(), 
        legend.position = &quot;none&quot;) +
    theme(...)
}

theme_paper &lt;-   function(...) {         # basic theme function to apply to all plots
  theme_min(axis.line.x = element_line(colour = &quot;black&quot;, size = 1),
            axis.line.y = element_line(colour = &quot;black&quot;, size = 1),
            axis.text.x = element_text(face = &quot;plain&quot;, colour = &quot;black&quot;, size = 16, angle = 0),
            axis.text.y = element_text(face = &quot;plain&quot;, colour = &quot;black&quot;, size = 16, angle = 0),
            axis.title.x = element_text(face = &quot;plain&quot;, size = 16, margin = margin(r = 0, 10, 0, 0)),
            axis.title.y = element_text(face = &quot;plain&quot;, size = 16, margin = margin(r = 10, 0, 0, 0)),
            axis.ticks = element_line(size = 1, colour = &quot;black&quot;),
            axis.ticks.length = unit(.3, &quot;cm&quot;)) +
    theme(...)
}

theme_density &lt;- function(...) {          # basic theme + theme specific to density plots
  theme_paper(legend.position = c(0.8, 0.975),
              legend.title = element_blank(),
              legend.text = element_text(colour = &quot;black&quot;, size = 16),
              legend.key = element_rect(colour = &quot;transparent&quot;, fill = &quot;transparent&quot;)) +
    theme(...)
}


theme_jitter &lt;- function(...) {           # density theme + theme specific to jitter plots
  theme_density(legend.position = &quot;none&quot;,
               axis.ticks = element_line(colour = &quot;transparent&quot;),
               axis.text.y = element_text(colour = &quot;transparent&quot;)) +
    theme(...)
}

draw_key_spaced &lt;- function(data, params, size) {  # resizes legend keys
  rectGrob(width = unit(0.8, &quot;npc&quot;),
           height = unit(0.8, &quot;npc&quot;),
           gp = gpar(col = data$colour,
                     fill = alpha(data$fill, data$alpha)))
}
GeomDensity$draw_key &lt;- draw_key_spaced</code></pre>
<div id="data" class="section level1">
<h1>Data</h1>
<p>To address the question whether sexual selection in females is widespread, we screened published papers to find female Bateman gradients (see Figure S5 and Materials and Methods). The dataset includes 111 effect sizes, extracted from 78 studies and across 72 species. <br></br></p>
<p><strong>Source Data</strong> Table of effect sizes included in the meta-analysis.</p>
<pre class="r"><code>Data &lt;- read.table(&quot;./data/Fromonteil_et_al_DATA.txt&quot;, fileEncoding = &quot;LATIN1&quot;) # the file encoding supports special characters
Tree &lt;- read.tree(&quot;./data/Fromonteil_et_al_TREE.txt&quot;)

Data %&gt;%
  dplyr::select(-c(4, 7, 14)) %&gt;% 
  mutate_if(is.numeric, round, digits = 3) %&gt;%
kable(&quot;html&quot;) %&gt;%
  kable_paper(full_width = T) %&gt;%
  scroll_box(width = &quot;100%&quot;, height = &quot;500px&quot;) %&gt;%
  kable_styling(fixed_thead = T)</code></pre>
<div class="table" style="border: 1px solid #ddd; padding: 0px; overflow-y: scroll; height:500px; overflow-x: scroll; width:100%;  margin-left: auto; margin-right: auto;">
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
Index
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
Study_ID
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
Study
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
Species_Phylo
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
Class
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
Polyandry
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
MatingSystem
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
StudyType
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
MS_estimate
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
Zero_f
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
beta_ss
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
r_f
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
r_VAR_f
</th>
<th style="text-align:right;position: sticky; top:0; background-color: #FFFFFF;position: sticky; top:0; background-color: #FFFFFF;">
N_f
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2
</td>
<td style="text-align:left;">
Gerlach et al. 2012
</td>
<td style="text-align:left;">
Junco_hyemalis
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.439
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.246
</td>
<td style="text-align:right;">
0.303
</td>
<td style="text-align:right;">
0.002
</td>
<td style="text-align:right;">
436
</td>
</tr>
<tr>
<td style="text-align:right;">
2
</td>
<td style="text-align:right;">
3
</td>
<td style="text-align:left;">
Gopurenko et al 2007
</td>
<td style="text-align:left;">
Ambystoma_texanum
</td>
<td style="text-align:left;">
Amphibia
</td>
<td style="text-align:right;">
0.857
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
18.342
</td>
<td style="text-align:right;">
0.817
</td>
<td style="text-align:right;">
0.003
</td>
<td style="text-align:right;">
37
</td>
</tr>
<tr>
<td style="text-align:right;">
4
</td>
<td style="text-align:right;">
4
</td>
<td style="text-align:left;">
Gopurenko et al. 2006
</td>
<td style="text-align:left;">
Ambystoma_tigrinum
</td>
<td style="text-align:left;">
Amphibia
</td>
<td style="text-align:right;">
0.467
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.078
</td>
<td style="text-align:right;">
0.065
</td>
<td style="text-align:right;">
0.071
</td>
<td style="text-align:right;">
15
</td>
</tr>
<tr>
<td style="text-align:right;">
5
</td>
<td style="text-align:right;">
9
</td>
<td style="text-align:left;">
Jones et al. 2002
</td>
<td style="text-align:left;">
Taricha_granulosa
</td>
<td style="text-align:left;">
Amphibia
</td>
<td style="text-align:right;">
0.548
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
22.200
</td>
<td style="text-align:right;">
0.269
</td>
<td style="text-align:right;">
0.030
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:right;">
6
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:left;">
Jones et al. 2004
</td>
<td style="text-align:left;">
Taricha_granulosa
</td>
<td style="text-align:left;">
Amphibia
</td>
<td style="text-align:right;">
0.548
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
21.700
</td>
<td style="text-align:right;">
0.456
</td>
<td style="text-align:right;">
0.126
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:right;">
7
</td>
<td style="text-align:right;">
10
</td>
<td style="text-align:left;">
Jones et al. 2004
</td>
<td style="text-align:left;">
Taricha_granulosa
</td>
<td style="text-align:left;">
Amphibia
</td>
<td style="text-align:right;">
0.548
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
-18.700
</td>
<td style="text-align:right;">
-0.206
</td>
<td style="text-align:right;">
0.183
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:right;">
8
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:left;">
Jones et al. 2000
</td>
<td style="text-align:left;">
Syngnathus_typhle
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.528
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
24.100
</td>
<td style="text-align:right;">
0.834
</td>
<td style="text-align:right;">
0.002
</td>
<td style="text-align:right;">
62
</td>
</tr>
<tr>
<td style="text-align:right;">
9
</td>
<td style="text-align:right;">
11
</td>
<td style="text-align:left;">
Jones et al. 2000
</td>
<td style="text-align:left;">
Syngnathus_typhle
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.528
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
14.500
</td>
<td style="text-align:right;">
0.588
</td>
<td style="text-align:right;">
0.039
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:right;">
10
</td>
<td style="text-align:right;">
14
</td>
<td style="text-align:left;">
Krakauer 2008
</td>
<td style="text-align:left;">
Meleagris_gallopavo
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.450
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2.299
</td>
<td style="text-align:right;">
0.336
</td>
<td style="text-align:right;">
0.028
</td>
<td style="text-align:right;">
29
</td>
</tr>
<tr>
<td style="text-align:right;">
11
</td>
<td style="text-align:right;">
16
</td>
<td style="text-align:left;">
Jones et al. 2012
</td>
<td style="text-align:left;">
Spermophilus_columbianus
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.343
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.280
</td>
<td style="text-align:right;">
0.225
</td>
<td style="text-align:right;">
0.009
</td>
<td style="text-align:right;">
99
</td>
</tr>
<tr>
<td style="text-align:right;">
12
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:left;">
Levitan 2008
</td>
<td style="text-align:left;">
Strongylocentrotus_franciscanus
</td>
<td style="text-align:left;">
Echinoidea
</td>
<td style="text-align:right;">
0.990
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.007
</td>
<td style="text-align:right;">
0.025
</td>
<td style="text-align:right;">
0.006
</td>
<td style="text-align:right;">
178
</td>
</tr>
<tr>
<td style="text-align:right;">
13
</td>
<td style="text-align:right;">
17
</td>
<td style="text-align:left;">
Levitan 2008
</td>
<td style="text-align:left;">
Strongylocentrotus_purpuratus
</td>
<td style="text-align:left;">
Echinoidea
</td>
<td style="text-align:right;">
0.990
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
-0.040
</td>
<td style="text-align:right;">
-0.115
</td>
<td style="text-align:right;">
0.020
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:right;">
14
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:left;">
Mills et al. 2007
</td>
<td style="text-align:left;">
Myodes_glareolus
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.353
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.200
</td>
<td style="text-align:right;">
0.175
</td>
<td style="text-align:right;">
0.085
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:right;">
15
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:left;">
Mills et al. 2007
</td>
<td style="text-align:left;">
Myodes_glareolus
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.353
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
-0.250
</td>
<td style="text-align:right;">
-0.180
</td>
<td style="text-align:right;">
0.134
</td>
<td style="text-align:right;">
8
</td>
</tr>
<tr>
<td style="text-align:right;">
16
</td>
<td style="text-align:right;">
19
</td>
<td style="text-align:left;">
Mills et al. 2007
</td>
<td style="text-align:left;">
Myodes_glareolus
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.353
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
0.010
</td>
<td style="text-align:right;">
0.200
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:right;">
17
</td>
<td style="text-align:right;">
20
</td>
<td style="text-align:left;">
Mobley &amp; Jones 2013
</td>
<td style="text-align:left;">
Syngnathus_floridae
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.633
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.930
</td>
<td style="text-align:right;">
0.897
</td>
<td style="text-align:right;">
0.001
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:right;">
19
</td>
<td style="text-align:right;">
21
</td>
<td style="text-align:left;">
Munroe &amp; Koprowski 2011
</td>
<td style="text-align:left;">
Xerospermophilus_tereticaudus
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.935
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.810
</td>
<td style="text-align:right;">
0.883
</td>
<td style="text-align:right;">
0.002
</td>
<td style="text-align:right;">
31
</td>
</tr>
<tr>
<td style="text-align:right;">
21
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:left;">
Moorad et al. 2011
</td>
<td style="text-align:left;">
Homo_sapiens
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.773
</td>
<td style="text-align:right;">
0.011
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
21865
</td>
</tr>
<tr>
<td style="text-align:right;">
22
</td>
<td style="text-align:right;">
22
</td>
<td style="text-align:left;">
Moorad et al. 2011
</td>
<td style="text-align:left;">
Homo_sapiens
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.332
</td>
<td style="text-align:right;">
0.008
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
6731
</td>
</tr>
<tr>
<td style="text-align:right;">
23
</td>
<td style="text-align:right;">
23
</td>
<td style="text-align:left;">
Borgerhoff Mulder 2009
</td>
<td style="text-align:left;">
Homo_sapiens
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.196
</td>
<td style="text-align:right;">
0.052
</td>
<td style="text-align:right;">
0.007
</td>
<td style="text-align:right;">
154
</td>
</tr>
<tr>
<td style="text-align:right;">
24
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:left;">
Croshaw 2010
</td>
<td style="text-align:left;">
Ambystoma_opacum
</td>
<td style="text-align:left;">
Amphibia
</td>
<td style="text-align:right;">
0.294
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
11.300
</td>
<td style="text-align:right;">
0.158
</td>
<td style="text-align:right;">
0.043
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:right;">
25
</td>
<td style="text-align:right;">
24
</td>
<td style="text-align:left;">
Croshaw 2010
</td>
<td style="text-align:left;">
Ambystoma_opacum
</td>
<td style="text-align:left;">
Amphibia
</td>
<td style="text-align:right;">
0.294
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
15.150
</td>
<td style="text-align:right;">
0.238
</td>
<td style="text-align:right;">
0.044
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:right;">
26
</td>
<td style="text-align:right;">
27
</td>
<td style="text-align:left;">
Schlicht &amp; Kempenears 2013
</td>
<td style="text-align:left;">
Cyanistes_caeruleus
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.470
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
-0.010
</td>
<td style="text-align:right;">
-0.002
</td>
<td style="text-align:right;">
0.002
</td>
<td style="text-align:right;">
447
</td>
</tr>
<tr>
<td style="text-align:right;">
27
</td>
<td style="text-align:right;">
28
</td>
<td style="text-align:left;">
Rodriguez-Munoz et al. 2010
</td>
<td style="text-align:left;">
Gryllus_campestris
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.809
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.840
</td>
<td style="text-align:right;">
0.133
</td>
<td style="text-align:right;">
0.018
</td>
<td style="text-align:right;">
55
</td>
</tr>
<tr>
<td style="text-align:right;">
28
</td>
<td style="text-align:right;">
29
</td>
<td style="text-align:left;">
Fritzsche &amp; Arnqvist 2013
</td>
<td style="text-align:left;">
Megabruchidius_tonkineus
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.977
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.872
</td>
<td style="text-align:right;">
0.755
</td>
<td style="text-align:right;">
0.004
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:right;">
29
</td>
<td style="text-align:right;">
29
</td>
<td style="text-align:left;">
Fritzsche &amp; Arnqvist 2013
</td>
<td style="text-align:left;">
Megabruchidius_dorsalis
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.617
</td>
<td style="text-align:right;">
0.610
</td>
<td style="text-align:right;">
0.008
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:right;">
30
</td>
<td style="text-align:right;">
29
</td>
<td style="text-align:left;">
Fritzsche &amp; Arnqvist 2013
</td>
<td style="text-align:left;">
Callosobruchus_maculatus
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.171
</td>
<td style="text-align:right;">
0.230
</td>
<td style="text-align:right;">
0.018
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:right;">
31
</td>
<td style="text-align:right;">
29
</td>
<td style="text-align:left;">
Fritzsche &amp; Arnqvist 2013
</td>
<td style="text-align:left;">
Callosobruchus_chinensis
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.680
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.096
</td>
<td style="text-align:right;">
0.195
</td>
<td style="text-align:right;">
0.019
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:right;">
32
</td>
<td style="text-align:right;">
31
</td>
<td style="text-align:left;">
Anthes et al. 2010
</td>
<td style="text-align:left;">
Biomphalaria_glabrata
</td>
<td style="text-align:left;">
Gastropoda
</td>
<td style="text-align:right;">
0.654
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.226
</td>
<td style="text-align:right;">
0.364
</td>
<td style="text-align:right;">
0.026
</td>
<td style="text-align:right;">
30
</td>
</tr>
<tr>
<td style="text-align:right;">
34
</td>
<td style="text-align:right;">
32
</td>
<td style="text-align:left;">
Ketterson et al. 1997
</td>
<td style="text-align:left;">
Junco_hyemalis
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.439
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
2.920
</td>
<td style="text-align:right;">
0.769
</td>
<td style="text-align:right;">
0.004
</td>
<td style="text-align:right;">
45
</td>
</tr>
<tr>
<td style="text-align:right;">
35
</td>
<td style="text-align:right;">
36
</td>
<td style="text-align:left;">
Garcia-Navas et al. 2014
</td>
<td style="text-align:left;">
Cyanistes_caeruleus
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.470
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.117
</td>
<td style="text-align:right;">
0.345
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:right;">
65
</td>
</tr>
<tr>
<td style="text-align:right;">
36
</td>
<td style="text-align:right;">
38
</td>
<td style="text-align:left;">
Gagnon et. al. 2012
</td>
<td style="text-align:left;">
Gerris_gillettei
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.776
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.050
</td>
<td style="text-align:right;">
0.365
</td>
<td style="text-align:right;">
0.008
</td>
<td style="text-align:right;">
97
</td>
</tr>
<tr>
<td style="text-align:right;">
37
</td>
<td style="text-align:right;">
39
</td>
<td style="text-align:left;">
Walker et al. 2014
</td>
<td style="text-align:left;">
Notiomystis_cincta
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.836
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.600
</td>
<td style="text-align:right;">
0.430
</td>
<td style="text-align:right;">
0.007
</td>
<td style="text-align:right;">
91
</td>
</tr>
<tr>
<td style="text-align:right;">
39
</td>
<td style="text-align:right;">
1001
</td>
<td style="text-align:left;">
Bergeron et al. 2012
</td>
<td style="text-align:left;">
Tamias_striatus
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.650
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:right;">
0.941
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
145
</td>
</tr>
<tr>
<td style="text-align:right;">
41
</td>
<td style="text-align:right;">
1002
</td>
<td style="text-align:left;">
Courtiol et al. 2012
</td>
<td style="text-align:left;">
Homo_sapiens
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.565
</td>
<td style="text-align:right;">
0.369
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
1669
</td>
</tr>
<tr>
<td style="text-align:right;">
43
</td>
<td style="text-align:right;">
1004
</td>
<td style="text-align:left;">
Andrade &amp; Kasumovic 2005
</td>
<td style="text-align:left;">
Latrodectus_hasseltii
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.667
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.475
</td>
<td style="text-align:right;">
0.394
</td>
<td style="text-align:right;">
0.006
</td>
<td style="text-align:right;">
119
</td>
</tr>
<tr>
<td style="text-align:right;">
46
</td>
<td style="text-align:right;">
1005
</td>
<td style="text-align:left;">
Balenger et al. 2009
</td>
<td style="text-align:left;">
Sialia_currucoides
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.342
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.034
</td>
<td style="text-align:right;">
0.067
</td>
<td style="text-align:right;">
0.017
</td>
<td style="text-align:right;">
59
</td>
</tr>
<tr>
<td style="text-align:right;">
47
</td>
<td style="text-align:right;">
1006
</td>
<td style="text-align:left;">
Barreto &amp; Avise 2010
</td>
<td style="text-align:left;">
Pycnogonum_stearnsi
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.686
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.990
</td>
<td style="text-align:right;">
0.877
</td>
<td style="text-align:right;">
0.001
</td>
<td style="text-align:right;">
62
</td>
</tr>
<tr>
<td style="text-align:right;">
48
</td>
<td style="text-align:right;">
1007
</td>
<td style="text-align:left;">
Becher &amp; Magurran 2004
</td>
<td style="text-align:left;">
Poecilia_reticulata
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.710
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.807
</td>
<td style="text-align:right;">
0.704
</td>
<td style="text-align:right;">
0.003
</td>
<td style="text-align:right;">
99
</td>
</tr>
<tr>
<td style="text-align:right;">
50
</td>
<td style="text-align:right;">
1008
</td>
<td style="text-align:left;">
Bjork &amp; Pitnick 2006
</td>
<td style="text-align:left;">
Drosophila_lummei
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.753
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.588
</td>
<td style="text-align:right;">
0.482
</td>
<td style="text-align:right;">
0.007
</td>
<td style="text-align:right;">
85
</td>
</tr>
<tr>
<td style="text-align:right;">
51
</td>
<td style="text-align:right;">
1008
</td>
<td style="text-align:left;">
Bjork &amp; Pitnick 2006
</td>
<td style="text-align:left;">
Drosophila_melanogaster
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.759
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.042
</td>
<td style="text-align:right;">
0.079
</td>
<td style="text-align:right;">
0.007
</td>
<td style="text-align:right;">
140
</td>
</tr>
<tr>
<td style="text-align:right;">
52
</td>
<td style="text-align:right;">
1008
</td>
<td style="text-align:left;">
Bjork &amp; Pitnick 2006
</td>
<td style="text-align:left;">
Drosophila_bifurca
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.326
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.237
</td>
<td style="text-align:right;">
0.201
</td>
<td style="text-align:right;">
0.008
</td>
<td style="text-align:right;">
114
</td>
</tr>
<tr>
<td style="text-align:right;">
53
</td>
<td style="text-align:right;">
1008
</td>
<td style="text-align:left;">
Bjork &amp; Pitnick 2006
</td>
<td style="text-align:left;">
Drosophila_virilis
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.753
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
-0.159
</td>
<td style="text-align:right;">
-0.221
</td>
<td style="text-align:right;">
0.022
</td>
<td style="text-align:right;">
42
</td>
</tr>
<tr>
<td style="text-align:right;">
54
</td>
<td style="text-align:right;">
1009
</td>
<td style="text-align:left;">
Collet et al. 2012
</td>
<td style="text-align:left;">
Gallus_gallus
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.950
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.290
</td>
<td style="text-align:right;">
0.608
</td>
<td style="text-align:right;">
0.033
</td>
<td style="text-align:right;">
13
</td>
</tr>
<tr>
<td style="text-align:right;">
55
</td>
<td style="text-align:right;">
1010
</td>
<td style="text-align:left;">
Fitze &amp; Le Galliard 2011
</td>
<td style="text-align:left;">
Zootoca_vivipara
</td>
<td style="text-align:left;">
Reptilia
</td>
<td style="text-align:right;">
0.633
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.482
</td>
<td style="text-align:right;">
0.950
</td>
<td style="text-align:right;">
0.002
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:right;">
56
</td>
<td style="text-align:right;">
1010
</td>
<td style="text-align:left;">
Fitze &amp; Le Galliard 2011
</td>
<td style="text-align:left;">
Zootoca_vivipara
</td>
<td style="text-align:left;">
Reptilia
</td>
<td style="text-align:right;">
0.633
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.947
</td>
<td style="text-align:right;">
0.890
</td>
<td style="text-align:right;">
0.009
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:right;">
59
</td>
<td style="text-align:right;">
1011
</td>
<td style="text-align:left;">
Jokela et al. 2010
</td>
<td style="text-align:left;">
Homo_sapiens
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.120
</td>
<td style="text-align:right;">
0.089
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
4010
</td>
</tr>
<tr>
<td style="text-align:right;">
60
</td>
<td style="text-align:right;">
1013
</td>
<td style="text-align:left;">
Aronsen et al. 2013
</td>
<td style="text-align:left;">
Syngnathus_typhle
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.528
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.956
</td>
<td style="text-align:right;">
0.977
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:right;">
61
</td>
<td style="text-align:right;">
1013
</td>
<td style="text-align:left;">
Aronsen et al. 2013
</td>
<td style="text-align:left;">
Syngnathus_typhle
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.528
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.987
</td>
<td style="text-align:right;">
0.895
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
100
</td>
</tr>
<tr>
<td style="text-align:right;">
62
</td>
<td style="text-align:right;">
1013
</td>
<td style="text-align:left;">
Aronsen et al. 2013
</td>
<td style="text-align:left;">
Syngnathus_typhle
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.528
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.855
</td>
<td style="text-align:right;">
0.729
</td>
<td style="text-align:right;">
0.002
</td>
<td style="text-align:right;">
100
</td>
</tr>
<tr>
<td style="text-align:right;">
63
</td>
<td style="text-align:right;">
1013
</td>
<td style="text-align:left;">
Aronsen et al. 2013
</td>
<td style="text-align:left;">
Syngnathus_typhle
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.528
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.040
</td>
<td style="text-align:right;">
0.662
</td>
<td style="text-align:right;">
0.006
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:right;">
64
</td>
<td style="text-align:right;">
1014
</td>
<td style="text-align:left;">
Rose et al. 2013
</td>
<td style="text-align:left;">
Syngnathus_scovelli
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.615
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.915
</td>
<td style="text-align:right;">
0.893
</td>
<td style="text-align:right;">
0.001
</td>
<td style="text-align:right;">
56
</td>
</tr>
<tr>
<td style="text-align:right;">
66
</td>
<td style="text-align:right;">
1017
</td>
<td style="text-align:left;">
Ursprung et al. 2011
</td>
<td style="text-align:left;">
Allobates_femoralis
</td>
<td style="text-align:left;">
Amphibia
</td>
<td style="text-align:right;">
0.571
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.770
</td>
<td style="text-align:right;">
0.542
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:right;">
42
</td>
</tr>
<tr>
<td style="text-align:right;">
68
</td>
<td style="text-align:right;">
2001
</td>
<td style="text-align:left;">
Pélissié et al. 2012
</td>
<td style="text-align:left;">
Physa_acuta
</td>
<td style="text-align:left;">
Gastropoda
</td>
<td style="text-align:right;">
0.789
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.336
</td>
<td style="text-align:right;">
0.260
</td>
<td style="text-align:right;">
0.007
</td>
<td style="text-align:right;">
120
</td>
</tr>
<tr>
<td style="text-align:right;">
71
</td>
<td style="text-align:right;">
2002
</td>
<td style="text-align:left;">
Pongratz &amp; Michiels 2003
</td>
<td style="text-align:left;">
Schmidtea_polychroa
</td>
<td style="text-align:left;">
Turbellaria
</td>
<td style="text-align:right;">
0.918
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.965
</td>
<td style="text-align:right;">
0.779
</td>
<td style="text-align:right;">
0.003
</td>
<td style="text-align:right;">
54
</td>
</tr>
<tr>
<td style="text-align:right;">
73
</td>
<td style="text-align:right;">
2003
</td>
<td style="text-align:left;">
Poesel et al. 2011
</td>
<td style="text-align:left;">
Zonotrichia_leucophrys
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.346
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.151
</td>
<td style="text-align:right;">
0.196
</td>
<td style="text-align:right;">
0.018
</td>
<td style="text-align:right;">
52
</td>
</tr>
<tr>
<td style="text-align:right;">
74
</td>
<td style="text-align:right;">
2004
</td>
<td style="text-align:left;">
Prosser et al. 2002
</td>
<td style="text-align:left;">
Nerodia_sipedon
</td>
<td style="text-align:left;">
Reptilia
</td>
<td style="text-align:right;">
0.556
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.789
</td>
<td style="text-align:right;">
0.381
</td>
<td style="text-align:right;">
0.017
</td>
<td style="text-align:right;">
45
</td>
</tr>
<tr>
<td style="text-align:right;">
76
</td>
<td style="text-align:right;">
2005
</td>
<td style="text-align:left;">
Rios-Cardenas 2005
</td>
<td style="text-align:left;">
Lepomis_gibbosus
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.244
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
64.170
</td>
<td style="text-align:right;">
0.031
</td>
<td style="text-align:right;">
0.008
</td>
<td style="text-align:right;">
123
</td>
</tr>
<tr>
<td style="text-align:right;">
77
</td>
<td style="text-align:right;">
2006
</td>
<td style="text-align:left;">
Schulte-Hostedde et al. 2004
</td>
<td style="text-align:left;">
Tamias_amoenus
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.595
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.857
</td>
<td style="text-align:right;">
0.870
</td>
<td style="text-align:right;">
0.002
</td>
<td style="text-align:right;">
37
</td>
</tr>
<tr>
<td style="text-align:right;">
79
</td>
<td style="text-align:right;">
2008
</td>
<td style="text-align:left;">
Tatarenkov et al. 2008
</td>
<td style="text-align:left;">
Xiphophorus_helleri
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.638
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.407
</td>
<td style="text-align:right;">
0.231
</td>
<td style="text-align:right;">
0.022
</td>
<td style="text-align:right;">
42
</td>
</tr>
<tr>
<td style="text-align:right;">
81
</td>
<td style="text-align:right;">
2008
</td>
<td style="text-align:left;">
Tatarenkov et al. 2008
</td>
<td style="text-align:left;">
Xiphophorus_helleri
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.638
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.234
</td>
<td style="text-align:right;">
0.152
</td>
<td style="text-align:right;">
0.037
</td>
<td style="text-align:right;">
27
</td>
</tr>
<tr>
<td style="text-align:right;">
83
</td>
<td style="text-align:right;">
2009
</td>
<td style="text-align:left;">
Whittingham &amp; Dunn 2005
</td>
<td style="text-align:left;">
Geothlypis_trichas
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.654
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.757
</td>
<td style="text-align:right;">
0.323
</td>
<td style="text-align:right;">
0.045
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:right;">
84
</td>
<td style="text-align:right;">
2010
</td>
<td style="text-align:left;">
Williams &amp; DeWoody 2009
</td>
<td style="text-align:left;">
Ambystoma_tigrinum
</td>
<td style="text-align:left;">
Amphibia
</td>
<td style="text-align:right;">
0.467
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.036
</td>
<td style="text-align:right;">
0.751
</td>
<td style="text-align:right;">
0.004
</td>
<td style="text-align:right;">
45
</td>
</tr>
<tr>
<td style="text-align:right;">
85
</td>
<td style="text-align:right;">
2011
</td>
<td style="text-align:left;">
Woolfenden et al. 2002
</td>
<td style="text-align:left;">
Molothrus_ater
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.455
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.910
</td>
<td style="text-align:right;">
0.700
</td>
<td style="text-align:right;">
0.002
</td>
<td style="text-align:right;">
142
</td>
</tr>
<tr>
<td style="text-align:right;">
86
</td>
<td style="text-align:right;">
2014
</td>
<td style="text-align:left;">
Serbezov et al. 2010
</td>
<td style="text-align:left;">
Salmo_trutta
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.680
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.026
</td>
<td style="text-align:right;">
0.970
</td>
<td style="text-align:right;">
0.000
</td>
<td style="text-align:right;">
116
</td>
</tr>
<tr>
<td style="text-align:right;">
88
</td>
<td style="text-align:right;">
2014
</td>
<td style="text-align:left;">
Serbezov et al. 2010
</td>
<td style="text-align:left;">
Salmo_trutta
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.680
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.967
</td>
<td style="text-align:right;">
0.718
</td>
<td style="text-align:right;">
0.002
</td>
<td style="text-align:right;">
121
</td>
</tr>
<tr>
<td style="text-align:right;">
89
</td>
<td style="text-align:right;">
2014
</td>
<td style="text-align:left;">
Serbezov et al. 2010
</td>
<td style="text-align:left;">
Salmo_trutta
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.680
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
1.117
</td>
<td style="text-align:right;">
0.576
</td>
<td style="text-align:right;">
0.004
</td>
<td style="text-align:right;">
107
</td>
</tr>
<tr>
<td style="text-align:right;">
90
</td>
<td style="text-align:right;">
2015
</td>
<td style="text-align:left;">
Whittingham &amp; Lifjeld 1995
</td>
<td style="text-align:left;">
Delichon_urbica
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.235
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
-0.057
</td>
<td style="text-align:right;">
-0.092
</td>
<td style="text-align:right;">
0.076
</td>
<td style="text-align:right;">
14
</td>
</tr>
<tr>
<td style="text-align:right;">
91
</td>
<td style="text-align:right;">
2016
</td>
<td style="text-align:left;">
Broquet et al. 2009
</td>
<td style="text-align:left;">
Hyla_arborea
</td>
<td style="text-align:left;">
Amphibia
</td>
<td style="text-align:right;">
0.158
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.438
</td>
<td style="text-align:right;">
0.159
</td>
<td style="text-align:right;">
0.053
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:right;">
93
</td>
<td style="text-align:right;">
2020
</td>
<td style="text-align:left;">
Byers et al. 2004
</td>
<td style="text-align:left;">
Dendroica_pensylvanica
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.606
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.686
</td>
<td style="text-align:right;">
0.523
</td>
<td style="text-align:right;">
0.015
</td>
<td style="text-align:right;">
37
</td>
</tr>
<tr>
<td style="text-align:right;">
94
</td>
<td style="text-align:right;">
3001
</td>
<td style="text-align:left;">
Buerkli &amp; Jokela 2017
</td>
<td style="text-align:left;">
Radix_balthica
</td>
<td style="text-align:left;">
Gastropoda
</td>
<td style="text-align:right;">
0.500
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
-0.010
</td>
<td style="text-align:right;">
-0.173
</td>
<td style="text-align:right;">
0.016
</td>
<td style="text-align:right;">
60
</td>
</tr>
<tr>
<td style="text-align:right;">
96
</td>
<td style="text-align:right;">
3003
</td>
<td style="text-align:left;">
Cattelan et al. 2020
</td>
<td style="text-align:left;">
Poecilia_reticulata
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.710
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.175
</td>
<td style="text-align:right;">
0.121
</td>
<td style="text-align:right;">
0.016
</td>
<td style="text-align:right;">
60
</td>
</tr>
<tr>
<td style="text-align:right;">
98
</td>
<td style="text-align:right;">
3003
</td>
<td style="text-align:left;">
Cattelan et al. 2020
</td>
<td style="text-align:left;">
Poecilia_reticulata
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.710
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.198
</td>
<td style="text-align:right;">
0.135
</td>
<td style="text-align:right;">
0.019
</td>
<td style="text-align:right;">
53
</td>
</tr>
<tr>
<td style="text-align:right;">
100
</td>
<td style="text-align:right;">
3004
</td>
<td style="text-align:left;">
Grunst et al. 2019
</td>
<td style="text-align:left;">
Zonotrichia_albicollis
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.638
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.193
</td>
<td style="text-align:right;">
0.180
</td>
<td style="text-align:right;">
0.004
</td>
<td style="text-align:right;">
222
</td>
</tr>
<tr>
<td style="text-align:right;">
101
</td>
<td style="text-align:right;">
3005
</td>
<td style="text-align:left;">
Hoffer et al. 2017
</td>
<td style="text-align:left;">
Lymnaea_stagnalis
</td>
<td style="text-align:left;">
Gastropoda
</td>
<td style="text-align:right;">
0.588
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
-0.083
</td>
<td style="text-align:right;">
-0.082
</td>
<td style="text-align:right;">
0.047
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:right;">
102
</td>
<td style="text-align:right;">
3006
</td>
<td style="text-align:left;">
Johannesson et al. 2016
</td>
<td style="text-align:left;">
Littorina_saxatilis
</td>
<td style="text-align:left;">
Gastropoda
</td>
<td style="text-align:right;">
0.680
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.055
</td>
<td style="text-align:right;">
0.071
</td>
<td style="text-align:right;">
0.041
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:right;">
105
</td>
<td style="text-align:right;">
3007
</td>
<td style="text-align:left;">
Devost &amp; Turgeon 2016
</td>
<td style="text-align:left;">
Gerris_buenoi
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
-0.416
</td>
<td style="text-align:right;">
-0.203
</td>
<td style="text-align:right;">
0.027
</td>
<td style="text-align:right;">
35
</td>
</tr>
<tr>
<td style="text-align:right;">
106
</td>
<td style="text-align:right;">
3008
</td>
<td style="text-align:left;">
Scheepers &amp; Gouws 2019
</td>
<td style="text-align:left;">
Clinus_cottoides
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.826
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.440
</td>
<td style="text-align:right;">
0.265
</td>
<td style="text-align:right;">
0.039
</td>
<td style="text-align:right;">
23
</td>
</tr>
<tr>
<td style="text-align:right;">
107
</td>
<td style="text-align:right;">
3009
</td>
<td style="text-align:left;">
Wang et al. 2020
</td>
<td style="text-align:left;">
Acanthopagrus_schlegelii
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.920
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.746
</td>
<td style="text-align:right;">
0.933
</td>
<td style="text-align:right;">
0.001
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:right;">
108
</td>
<td style="text-align:right;">
3010
</td>
<td style="text-align:left;">
Louder et al. 2019
</td>
<td style="text-align:left;">
Molothrus_ater
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.455
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.480
</td>
<td style="text-align:right;">
0.177
</td>
<td style="text-align:right;">
0.009
</td>
<td style="text-align:right;">
100
</td>
</tr>
<tr>
<td style="text-align:right;">
110
</td>
<td style="text-align:right;">
3011
</td>
<td style="text-align:left;">
Janicke et al. 2015
</td>
<td style="text-align:left;">
Physa_acuta
</td>
<td style="text-align:left;">
Gastropoda
</td>
<td style="text-align:right;">
0.789
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.543
</td>
<td style="text-align:right;">
0.534
</td>
<td style="text-align:right;">
0.013
</td>
<td style="text-align:right;">
39
</td>
</tr>
<tr>
<td style="text-align:right;">
112
</td>
<td style="text-align:right;">
3011
</td>
<td style="text-align:left;">
Janicke et al. 2015
</td>
<td style="text-align:left;">
Physa_acuta
</td>
<td style="text-align:left;">
Gastropoda
</td>
<td style="text-align:right;">
0.789
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.007
</td>
<td style="text-align:right;">
0.005
</td>
<td style="text-align:right;">
0.026
</td>
<td style="text-align:right;">
39
</td>
</tr>
<tr>
<td style="text-align:right;">
114
</td>
<td style="text-align:right;">
3012
</td>
<td style="text-align:left;">
Levine et al. 2015
</td>
<td style="text-align:left;">
Agkistrodon_contortrix
</td>
<td style="text-align:left;">
Reptilia
</td>
<td style="text-align:right;">
0.520
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.070
</td>
<td style="text-align:right;">
0.032
</td>
<td style="text-align:right;">
0.055
</td>
<td style="text-align:right;">
19
</td>
</tr>
<tr>
<td style="text-align:right;">
115
</td>
<td style="text-align:right;">
3013
</td>
<td style="text-align:left;">
Borgerhoff Mulder &amp; Ross 2019
</td>
<td style="text-align:left;">
Homo_sapiens
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.100
</td>
<td style="text-align:right;">
0.163
</td>
<td style="text-align:right;">
0.005
</td>
<td style="text-align:right;">
176
</td>
</tr>
<tr>
<td style="text-align:right;">
116
</td>
<td style="text-align:right;">
3014
</td>
<td style="text-align:left;">
Bolopo et al. 2016
</td>
<td style="text-align:left;">
Clamator_glandarius
</td>
<td style="text-align:left;">
Aves
</td>
<td style="text-align:right;">
0.697
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.080
</td>
<td style="text-align:right;">
0.727
</td>
<td style="text-align:right;">
0.003
</td>
<td style="text-align:right;">
65
</td>
</tr>
<tr>
<td style="text-align:right;">
117
</td>
<td style="text-align:right;">
3015
</td>
<td style="text-align:left;">
Glaudas et al. 2020
</td>
<td style="text-align:left;">
Bitis_arietans
</td>
<td style="text-align:left;">
Reptilia
</td>
<td style="text-align:right;">
0.654
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
2.480
</td>
<td style="text-align:right;">
0.386
</td>
<td style="text-align:right;">
0.045
</td>
<td style="text-align:right;">
17
</td>
</tr>
<tr>
<td style="text-align:right;">
118
</td>
<td style="text-align:right;">
3016
</td>
<td style="text-align:left;">
Saunders &amp; Shuster 2019
</td>
<td style="text-align:left;">
Paracerceis_sculpta
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.789
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.780
</td>
<td style="text-align:right;">
0.195
</td>
<td style="text-align:right;">
0.185
</td>
<td style="text-align:right;">
6
</td>
</tr>
<tr>
<td style="text-align:right;">
119
</td>
<td style="text-align:right;">
3017
</td>
<td style="text-align:left;">
Paczolt et al. 2015
</td>
<td style="text-align:left;">
Xiphophorus_birchmanni
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.839
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.702
</td>
<td style="text-align:right;">
0.613
</td>
<td style="text-align:right;">
0.013
</td>
<td style="text-align:right;">
31
</td>
</tr>
<tr>
<td style="text-align:right;">
120
</td>
<td style="text-align:right;">
3017
</td>
<td style="text-align:left;">
Paczolt et al. 2015
</td>
<td style="text-align:left;">
Xiphophorus_birchmanni
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.839
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.987
</td>
<td style="text-align:right;">
0.551
</td>
<td style="text-align:right;">
0.044
</td>
<td style="text-align:right;">
12
</td>
</tr>
<tr>
<td style="text-align:right;">
121
</td>
<td style="text-align:right;">
3018
</td>
<td style="text-align:left;">
Mangold et al. 2015
</td>
<td style="text-align:left;">
Hyalinobatrachium_valerioi
</td>
<td style="text-align:left;">
Amphibia
</td>
<td style="text-align:right;">
0.736
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.940
</td>
<td style="text-align:right;">
0.895
</td>
<td style="text-align:right;">
0.001
</td>
<td style="text-align:right;">
77
</td>
</tr>
<tr>
<td style="text-align:right;">
123
</td>
<td style="text-align:right;">
3019
</td>
<td style="text-align:left;">
Marie-Orleach et al. 2016
</td>
<td style="text-align:left;">
Macrostomum_lignano
</td>
<td style="text-align:left;">
Turbellaria
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.290
</td>
<td style="text-align:right;">
0.097
</td>
<td style="text-align:right;">
0.019
</td>
<td style="text-align:right;">
52
</td>
</tr>
<tr>
<td style="text-align:right;">
124
</td>
<td style="text-align:right;">
3020
</td>
<td style="text-align:left;">
Kretzschmar et al. 2019
</td>
<td style="text-align:left;">
Ceratotherium_simum
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.697
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.059
</td>
<td style="text-align:right;">
0.830
</td>
<td style="text-align:right;">
0.003
</td>
<td style="text-align:right;">
33
</td>
</tr>
<tr>
<td style="text-align:right;">
125
</td>
<td style="text-align:right;">
3021
</td>
<td style="text-align:left;">
Morimoto et al. 2016
</td>
<td style="text-align:left;">
Drosophila_melanogaster
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.759
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.020
</td>
<td style="text-align:right;">
0.010
</td>
<td style="text-align:right;">
0.018
</td>
<td style="text-align:right;">
58
</td>
</tr>
<tr>
<td style="text-align:right;">
126
</td>
<td style="text-align:right;">
3022
</td>
<td style="text-align:left;">
Turnell and Shaw 2015
</td>
<td style="text-align:left;">
Laupala_cerasina
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.197
</td>
<td style="text-align:right;">
0.292
</td>
<td style="text-align:right;">
0.042
</td>
<td style="text-align:right;">
21
</td>
</tr>
<tr>
<td style="text-align:right;">
127
</td>
<td style="text-align:right;">
3023
</td>
<td style="text-align:left;">
Fuxjäger et al. 2019
</td>
<td style="text-align:left;">
Gasterosteus_aculeatus
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.950
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.954
</td>
<td style="text-align:right;">
0.791
</td>
<td style="text-align:right;">
0.004
</td>
<td style="text-align:right;">
36
</td>
</tr>
<tr>
<td style="text-align:right;">
128
</td>
<td style="text-align:right;">
3023
</td>
<td style="text-align:left;">
Fuxjäger et al. 2019
</td>
<td style="text-align:left;">
Gasterosteus_aculeatus
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.950
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.922
</td>
<td style="text-align:right;">
0.686
</td>
<td style="text-align:right;">
0.008
</td>
<td style="text-align:right;">
36
</td>
</tr>
<tr>
<td style="text-align:right;">
131
</td>
<td style="text-align:right;">
3024
</td>
<td style="text-align:left;">
McCullough et al. 2018
</td>
<td style="text-align:left;">
Onthophagus_taurus
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.789
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.994
</td>
<td style="text-align:right;">
0.772
</td>
<td style="text-align:right;">
0.003
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:right;">
132
</td>
<td style="text-align:right;">
3024
</td>
<td style="text-align:left;">
McCullough et al. 2018
</td>
<td style="text-align:left;">
Onthophagus_taurus
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.789
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.693
</td>
<td style="text-align:right;">
0.636
</td>
<td style="text-align:right;">
0.004
</td>
<td style="text-align:right;">
89
</td>
</tr>
<tr>
<td style="text-align:right;">
133
</td>
<td style="text-align:right;">
3024
</td>
<td style="text-align:left;">
McCullough et al. 2018
</td>
<td style="text-align:left;">
Onthophagus_taurus
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.789
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
1.301
</td>
<td style="text-align:right;">
0.780
</td>
<td style="text-align:right;">
0.007
</td>
<td style="text-align:right;">
22
</td>
</tr>
<tr>
<td style="text-align:right;">
134
</td>
<td style="text-align:right;">
3025
</td>
<td style="text-align:left;">
Wells et al. 2017
</td>
<td style="text-align:left;">
Callospermophilus_lateralis
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.630
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.460
</td>
<td style="text-align:right;">
0.224
</td>
<td style="text-align:right;">
0.038
</td>
<td style="text-align:right;">
25
</td>
</tr>
<tr>
<td style="text-align:right;">
137
</td>
<td style="text-align:right;">
3028
</td>
<td style="text-align:left;">
Skjærvø &amp; Røskaft 2015
</td>
<td style="text-align:left;">
Homo_sapiens
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.160
</td>
<td style="text-align:right;">
0.144
</td>
<td style="text-align:right;">
0.001
</td>
<td style="text-align:right;">
753
</td>
</tr>
<tr>
<td style="text-align:right;">
138
</td>
<td style="text-align:right;">
3028
</td>
<td style="text-align:left;">
Skjærvø &amp; Røskaft 2015
</td>
<td style="text-align:left;">
Homo_sapiens
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.100
</td>
<td style="text-align:right;">
0.059
</td>
<td style="text-align:right;">
0.002
</td>
<td style="text-align:right;">
443
</td>
</tr>
<tr>
<td style="text-align:right;">
139
</td>
<td style="text-align:right;">
3028
</td>
<td style="text-align:left;">
Skjærvø &amp; Røskaft 2015
</td>
<td style="text-align:left;">
Homo_sapiens
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.020
</td>
<td style="text-align:right;">
0.017
</td>
<td style="text-align:right;">
0.003
</td>
<td style="text-align:right;">
395
</td>
</tr>
<tr>
<td style="text-align:right;">
140
</td>
<td style="text-align:right;">
3028
</td>
<td style="text-align:left;">
Skjærvø &amp; Røskaft 2015
</td>
<td style="text-align:left;">
Homo_sapiens
</td>
<td style="text-align:left;">
Mammalia
</td>
<td style="text-align:right;">
0.012
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
-0.020
</td>
<td style="text-align:right;">
-0.012
</td>
<td style="text-align:right;">
0.005
</td>
<td style="text-align:right;">
205
</td>
</tr>
<tr>
<td style="text-align:right;">
141
</td>
<td style="text-align:right;">
3029
</td>
<td style="text-align:left;">
Levine et al. 2020
</td>
<td style="text-align:left;">
Crotalus_atrox
</td>
<td style="text-align:left;">
Reptilia
</td>
<td style="text-align:right;">
0.400
</td>
<td style="text-align:left;">
monogamous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.151
</td>
<td style="text-align:right;">
0.099
</td>
<td style="text-align:right;">
0.058
</td>
<td style="text-align:right;">
18
</td>
</tr>
<tr>
<td style="text-align:right;">
142
</td>
<td style="text-align:right;">
3030
</td>
<td style="text-align:left;">
Fromonteil et al. 2021
</td>
<td style="text-align:left;">
Acanthoscelides_obtectus
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.551
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.489
</td>
<td style="text-align:right;">
0.390
</td>
<td style="text-align:right;">
0.015
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:right;">
144
</td>
<td style="text-align:right;">
3030
</td>
<td style="text-align:left;">
Fromonteil et al. 2021
</td>
<td style="text-align:left;">
Acanthoscelides_obtectus
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.551
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.910
</td>
<td style="text-align:right;">
0.525
</td>
<td style="text-align:right;">
0.011
</td>
<td style="text-align:right;">
50
</td>
</tr>
<tr>
<td style="text-align:right;">
146
</td>
<td style="text-align:right;">
3030
</td>
<td style="text-align:left;">
Fromonteil et al. 2021
</td>
<td style="text-align:left;">
Acanthoscelides_obtectus
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.551
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
0.688
</td>
<td style="text-align:right;">
0.308
</td>
<td style="text-align:right;">
0.015
</td>
<td style="text-align:right;">
55
</td>
</tr>
<tr>
<td style="text-align:right;">
147
</td>
<td style="text-align:right;">
3030
</td>
<td style="text-align:left;">
Fromonteil et al. 2021
</td>
<td style="text-align:left;">
Acanthoscelides_obtectus
</td>
<td style="text-align:left;">
Arthropoda
</td>
<td style="text-align:right;">
0.551
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Lab
</td>
<td style="text-align:left;">
cMS
</td>
<td style="text-align:right;">
1
</td>
<td style="text-align:right;">
0.025
</td>
<td style="text-align:right;">
0.020
</td>
<td style="text-align:right;">
0.019
</td>
<td style="text-align:right;">
55
</td>
</tr>
<tr>
<td style="text-align:right;">
149
</td>
<td style="text-align:right;">
3031
</td>
<td style="text-align:left;">
LaBrecque et al. 2014
</td>
<td style="text-align:left;">
Hyperprosopon_anale
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
1.000
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
0.478
</td>
<td style="text-align:right;">
0.026
</td>
<td style="text-align:right;">
24
</td>
</tr>
<tr>
<td style="text-align:right;">
150
</td>
<td style="text-align:right;">
3031
</td>
<td style="text-align:left;">
LaBrecque et al. 2014
</td>
<td style="text-align:left;">
Cymatogaster_aggregata
</td>
<td style="text-align:left;">
Actinopterygii
</td>
<td style="text-align:right;">
0.800
</td>
<td style="text-align:left;">
polyandrous
</td>
<td style="text-align:left;">
Field
</td>
<td style="text-align:left;">
gMS
</td>
<td style="text-align:right;">
0
</td>
<td style="text-align:right;">
NA
</td>
<td style="text-align:right;">
0.514
</td>
<td style="text-align:right;">
0.011
</td>
<td style="text-align:right;">
50
</td>
</tr>
</tbody>
</table>
</div>
<p><br></br> <strong>Index:</strong> Individual number given to each effect size (k = 111).<br />
<strong>Study_ID:</strong> Identification number given to each study included in the meta-analysis.<br />
<strong>Study:</strong> Authors and year of the published study.<br />
<strong>Species_Phylo:</strong> Species (n = 72).<br />
<strong>Polyandry:</strong> Level of polyandry of the species, bound between 0 (exclusively monogamous) and 1 (exclusively polyandrous).<br />
<strong>MatingSystem:</strong> Mating system of female individuals. Either monandrous or polyandrous.<br />
<strong>StudyType:</strong> Field study or laboratory study.<br />
<strong>MS_estimate:</strong> Mating success can be estimated either by using the number of genetic mating partners (<em>gMS</em>) or copulatory mating success (<em>cMS</em>).<br />
<strong>Zero_f:</strong> Whether this effect size was calculated on a population which includes (1) or excludes (0) individuals with zero mating success.<br />
<strong>beta_ss:</strong> Female Bateman gradients, as reported in the published studies. Only some Bateman gradients are standardized.<br />
<strong>r_f:</strong> Pearson correlation coefficients converted from female Bateman gradients. <strong>r_VAR_f:</strong> Variance of the effect size (r).<br />
<strong>N_f:</strong> Sample size of the study.<br />
<br> As only some of the extracted Bateman gradients were computed on relativised data, we converted all obtained slopes into Pearson correlation coefficients (<em>r</em>). By running linear regression models between the actual Bateman Gradient and the correlation coefficients, we can check whether <em>r</em> is a reliable estimate for the benefit of mating.</p>
<pre class="r"><code># Relationship between r and betaSS ####
Data_stand &lt;- subset(Data, beta_ss_stand == &quot;1&quot;)

LM_beta_f_r &lt;- lm(Data_stand$beta_ss ~ Data_stand$r_f)


Scatter_r_versus_betaSS &lt;- ggplot(data = Data_stand,
                                  mapping = aes(x = r_f, y = beta_ss)) +
  geom_hline(yintercept = 0, linetype = &quot;longdash&quot;, color = &quot;black&quot;, size = 0.5) +
  geom_vline(xintercept = 0, linetype = &quot;longdash&quot;, color = &quot;black&quot;, size = 0.5) +
  labs(title = &quot;&quot;, x = &quot;*r*&quot;, y = expression(italic(beta[ss]))) +
  geom_point(shape = 21, size = 5, fill = &quot;grey&quot;, alpha = .6) +
  scale_y_continuous(limits = c(-0.6, 1.4), breaks = c(-0.6, 0, 0.6, 1.2)) + 
  scale_x_continuous(limits = c(-0.3, 1.0), breaks = c(0, 0.5, 1.0)) + 
  geom_smooth(method = lm, se = TRUE, linetype = &quot;longdash&quot;, color = &quot;#008080&quot;, fill = &quot;#B2D8D8&quot;, size = 0.8) +
  theme_paper(axis.title.x = element_markdown()) +
  annotate(&quot;text&quot;, x = 0.8, y = -0.3, label = paste0(&quot;R^2 == &quot;, round(summary(LM_beta_f_r)$r.squared, 2)), parse = TRUE, size = 7)
Scatter_r_versus_betaSS</code></pre>
<p><img src="figure/META_SexSelFem.Rmd/unnamed-chunk-1-1.png" width="672" style="display: block; margin: auto;" /> <strong>Figure S7A.</strong> Relationship between effect size <em>r</em> (correlation coefficient) and the actual Bateman gradient (<span class="math inline">\(\beta\)</span><sub>ss</sub>). Graph shows a subset of data for which standardized Bateman gradients were available (<em>N</em> = 68). Dashed line shows the linear regression fit with shaded area indicating the 95% confidence intervals.</p>
<div id="phylogeny" class="section level2">
<h2>Phylogeny</h2>
<p>We reconstructed the phylogeny using divergence times extracted from the TimeTree database (<a href="http://www.timetree.org/" class="uri">http://www.timetree.org/</a>, <span class="citation">(Kumar et al. 2017)</span>) and by using the <code>ggtree</code> package <span class="citation">(Yu et al. 2017, 2018; Yu 2020)</span>.</p>
<pre class="r"><code>MetaData_Species &lt;- unique(Data$Species_Phylo)
theTree &lt;- drop.tip(Tree, Tree$tip.label[-na.omit(match(MetaData_Species, Tree$tip.label))])
theTree$tip.label &lt;- gsub(&quot;_&quot;, &quot; &quot;, theTree$tip.label)

# Phylogenetic tree ####
Phylo &lt;- ggtree(theTree, size = 0.6) +
  ylim(-2,76) +
  xlim(-20, 650) +
  annotate(&quot;rect&quot;, xmin = 368, xmax = 570, ymin = 72.5, ymax = 70.5, fill = Palette(9)[1], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 368, xmax = 570, ymin = 70.5, ymax = 65.5, fill = Palette(9)[2], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 368, xmax = 570, ymin = 65.5, ymax = 48.5, fill = Palette(9)[3], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 368, xmax = 570, ymin = 48.5, ymax = 46.5, fill = Palette(9)[4], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 368, xmax = 570, ymin = 46.5, ymax = 33.5, fill = Palette(9)[5], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 368, xmax = 570, ymin = 33.5, ymax = 26.5, fill = Palette(9)[6], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 368, xmax = 570, ymin = 26.5, ymax = 13.5, fill = Palette(9)[7], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 368, xmax = 570, ymin = 13.5, ymax = 8.5, fill = Palette(9)[8], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 368, xmax = 570, ymin = 8.5, ymax = 0, fill = Palette(9)[9], alpha = .8) +
   geom_tiplab(size = 3, colour = &quot;black&quot;, hjust = 0.0) +
   theme_tree()
Phylo &lt;- rotate(Phylo, 73) %&gt;% rotate(77) %&gt;% rotate(79) %&gt;% rotate(91) %&gt;% rotate(95) 

# Number of effect sizes/class ####
Data_r_f &lt;- as.data.frame(aggregate(r_f ~ Class + Species_Phylo + Index, data = Data, FUN = mean, na.rm = TRUE))
Data_r_by_Species &lt;- as.data.frame(aggregate(r_f ~ Data_r_f$Class + Data_r_f$Species, data = Data_r_f, FUN = length))
colnames(Data_r_by_Species) &lt;- c(&quot;Class&quot;, &quot;Species&quot;, &quot;N_f_r&quot;)
Data_r_by_Class_ES &lt;- as.data.frame(aggregate(N_f_r ~ Class, data = Data_r_by_Species, FUN = sum))
Data_r_by_Class_ES_Species &lt;- as.data.frame(aggregate(N_f_r ~ Class, data = Data_r_by_Species, FUN = length))

Data_r_by_Class_ES$N_species &lt;- Data_r_by_Class_ES_Species$N_f_r
Data_r_by_Class_ES$order      &lt;- c(5, 6, 3, 7, 4, 2, 9, 8, 1)
rownames(Data_r_by_Class_ES) &lt;- 1:nrow(Data_r_by_Class_ES)


ES_Sum &lt;- sum(Data_r_by_Class_ES$N_f_r)
DChart_N_ES &lt;- ggplot(data = Data_r_by_Class_ES, 
                      mapping = aes(x = 1, y = N_f_r, fill = as.factor(order))) +
  scale_fill_manual(values = Palette(9)) +
  geom_bar(stat = &quot;identity&quot;) +
  xlim(0, 1.5) +
  coord_polar(theta = &quot;y&quot;, direction = -1) +   # polar coordinate system
  labs(x = NULL, y = NULL) +
  labs(fill = &quot;&quot;) +
theme_min(plot.title = element_text(face = &quot;bold&quot;, family = &quot;sans&quot;, size = 4),
          plot.margin = margin(l = -0.5, unit = &quot;cm&quot;),
          legend.text = element_text(size = 10),
          axis.ticks = element_blank(),
          axis.text = element_blank(),
          axis.title = element_blank()) +
  annotate(&quot;text&quot;, x = 0, y = 1, label = ES_Sum, size = 5, fontface = &quot;bold&quot;, angle = 0) 

# Number of species/class ####
Species_Sum &lt;- sum(Data_r_by_Class_ES$N_species)
DChart_N_Species &lt;- ggplot(data = Data_r_by_Class_ES, 
                           mapping = aes(x = 1, y = N_species, fill = as.factor(order))) +
  scale_fill_manual(values = Palette(9)) +
  geom_bar(stat = &quot;identity&quot;) +
  xlim(0, 1.5) +
  coord_polar(theta = &quot;y&quot;, direction = -1) +   # polar coordinate system
  labs(x = NULL, y = NULL) +
  labs(fill = &quot;&quot;) +
theme_min(plot.title = element_text(face = &quot;bold&quot;, family = &quot;sans&quot;, size = 4),
          plot.margin = margin(l = -0.5, unit = &quot;cm&quot;),
          legend.text = element_text(size = 10),
          axis.ticks = element_blank(),
          axis.text = element_blank(),
          axis.title = element_blank()) +
  annotate(&quot;text&quot;, x = 0, y = 1, label = Species_Sum, size = 5, fontface = &quot;bold&quot;, angle = 0)

Figure_Donut_ALL &lt;- plot_grid(DChart_N_ES, DChart_N_Species,
                              label_size = 15,
                              hjust = 0, 
                              vjust = 0, 
                              align = &quot;hv&quot;,
                              ncol = 2)

# Combine 
ggdraw(Phylo) + draw_plot(Figure_Donut_ALL, x = - 0.33, y = -0.34, scale = 0.3) +
  annotate(&quot;text&quot;, x = 0.085, y = 0.08, label = &quot;Sample\nsize&quot;, size = 5, angle=0, lineheight = 1, hjust = 0.5) +
  annotate(&quot;text&quot;, x = 0.245, y = 0.08, label = &quot;Species\nnumber&quot;, size = 5, angle=0, lineheight = 1, hjust = 0.5)</code></pre>
<p><img src="figure/META_SexSelFem.Rmd/Phylogeny-1.png" width="768" style="display: block; margin: auto;" /> <strong>Figure S6.</strong> Phylogenetic tree of all sampled species. Doughnut charts show the relative fraction of the sampled effect sizes (i.e., number of Bateman gradients) and the number of species.</p>
</div>
<div id="forest-plot" class="section level2">
<h2>Forest plot</h2>
<pre class="r"><code>Data$r_SD &lt;- sqrt(Data$r_VAR_f) # standard deviation
Data$r_SE &lt;- Data$r_SD / sqrt(Data$N_f) # standard error

Data$l95CI_r_f &lt;- Data$r_f - ((Data$r_SE)*1.96) # 95% confidence interval - lower
Data$u95CI_r_f &lt;- Data$r_f + ((Data$r_SE)*1.96) # 95% confidence interval - upper 
ForestData &lt;- Data[order(Data$Phylo_ID), ] 
ForestData$Phylo_ID_red &lt;- seq.int(nrow(ForestData))

ForestPlot &lt;- ggplot() + 
  geom_point(data = ForestData,
             mapping = aes(x = r_f, y = Phylo_ID_red),
             shape = 18, size = 2, colour = &quot;black&quot;) +
  geom_errorbarh(data = ForestData, 
                 mapping = aes(y = Phylo_ID_red, xmin = l95CI_r_f, xmax = u95CI_r_f),
                 size = 0.7, colour = &quot;black&quot;, height = 0.0) +
  annotate(&quot;rect&quot;, xmin = 1.2, xmax = 2.2, ymin = 111.5, ymax = 109.5, fill = Palette(9)[1], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 1.2, xmax = 2.2, ymin = 109.5, ymax = 102.5, fill = Palette(9)[2], alpha = .8) + 
  annotate(&quot;rect&quot;, xmin = 1.2, xmax = 2.2, ymin = 102.5, ymax = 79.5, fill = Palette(9)[3], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 1.2, xmax = 2.2, ymin = 79.5, ymax = 77.5, fill = Palette(9)[4], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 1.2, xmax = 2.2, ymin = 77.5, ymax = 52.5, fill = Palette(9)[5], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 1.2, xmax = 2.2, ymin = 52.5, ymax = 41.5, fill = Palette(9)[6], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 1.2, xmax = 2.2, ymin = 41.5, ymax = 25.5, fill = Palette(9)[7], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 1.2, xmax = 2.2, ymin = 25.5, ymax = 19.5, fill = Palette(9)[8], alpha = .8) +
  annotate(&quot;rect&quot;, xmin = 1.2, xmax = 2.2, ymin = 19.5, ymax = 0, fill = Palette(9)[9], alpha = .8) +
  geom_text(data = ForestData,
            mapping = aes(label = Study, x = 2.2, y = Phylo_ID_red), 
            colour = &quot;black&quot;, size = 2, angle = 0, check_overlap = FALSE, hjust = &quot;right&quot;) + 
  geom_vline(xintercept = 0, linetype = &quot;dashed&quot;, colour = &quot;black&quot;, size = 1)+
  scale_x_continuous(limits = c(-1.2, 2.2), breaks = c(-0.5, 0, 0.5, 1), expand = c(0 ,0)) + 
  labs(title = &quot;&quot;, x = &quot;Female Bateman gradient (*r*)&quot;, y = &quot;Study ID&quot;) +
  theme_paper(axis.title.x = element_text(size = 18),
              axis.title.y = element_text(size = 18)) +
  theme(axis.title.x = element_markdown())
ForestPlot</code></pre>
<p><img src="figure/META_SexSelFem.Rmd/Forest_Plot-1.png" width="576" style="display: block; margin: auto;" /> <strong>Figure S2.</strong> Forest plot of all sampled effect sizes. Effect sizes (Pearson correlation coefficient of Bateman gradients) with 95% confidence limits are shown in phylogenetic order. <br></br> ______________<br />
# Bayesian Meta-Analysis We ran General Linear Mixed-Effects Models (GLMMs) with the <code>MCMCglmm::MCMCglmm</code> function <span class="citation">(Hadfield 2010)</span> to provide a global test for sexual selection in females and to explore determinants of the inter-study variation. We used uninformative priors (<em>V</em> = 1, <em>nu</em> = 0.002) and an effective sample size of 10,000 (number of iterations = 4,400,000, burn-in = 400,000, thinning interval = 400).</p>
<pre class="r"><code>pr  &lt;- list(R = list(V = 1, nu = 0.002), G = list(G1 = list(V = 1, nu = 0.002),
                                         G2 = list(V = 1, nu = 0.002)))

pr2 &lt;- list(R = list(V = 1, nu = 0.002), G = list(G1 = list(V = 1, nu = 0.002),
                                         G2 = list(V = 1, nu = 0.002),
                                         G3 = list(V = 1, nu = 0.002)))

BURNIN =   400 #000
NITT   = 4400 #000
THIN   =      400</code></pre>
<p>GLMMs were run with the effect size (r) defined as the response variable weighted by the inverse of its sampling size and included study identifier and observation identifier as a random term.</p>
</div>
<div id="global-effect-size-traditionnal-and-phylogenetic-approach" class="section level2">
<h2>Global effect size: traditionnal and phylogenetic approach</h2>
<p>First, we quantified global effect sizes both without (i.e., ‘non-phylogenetic’ GLMMs) and with adding the phylogenetic correlation matrix as an additional random term (‘phylogenetic’ GLMMs).</p>
<pre class="r"><code>names(Data)[names(Data) == &quot;Species_Phylo&quot;] &lt;- &quot;animal&quot; # MCMCglmmm requires the species names to be in a column named &#39;animal&#39;

# models ####
Model_ALL_traditional &lt;- MCMCglmm(r_f~1,
                                  random = ~Study_ID + Index, 
                                  mev = Data$r_VAR_f,
                                  data = Data,
                                  prior = pr, pr = TRUE, verbose = FALSE,
                                  burnin = BURNIN, nitt = NITT, thin = THIN)
Model_ALL_phylo       &lt;- MCMCglmm(r_f~1,
                                  random = ~animal + Study_ID + Index, 
                                  mev = Data$r_VAR_f,
                                  pedigree = Tree,
                                  data = Data,
                                  prior = pr2, pr = TRUE, verbose = FALSE,
                                  burnin = BURNIN, nitt = NITT, thin = THIN)

# Raincloud plot ####
  # density
Data_trad  &lt;- as.data.frame(Model_ALL_traditional$Sol[,&quot;(Intercept)&quot;])
Data_trad$Approach &lt;- as.factor(rep(&quot;Non-phylogenetic&quot;, nrow(Data_trad)))
colnames(Data_trad) &lt;- c(&quot;ES&quot;, &quot;Approach&quot;)

Data_phylo  &lt;- as.data.frame(Model_ALL_phylo$Sol[,&quot;(Intercept)&quot;])
Data_phylo$Approach &lt;- as.factor(rep(&quot;Phylogenetic&quot;, nrow(Data_phylo)))
colnames(Data_phylo) &lt;- c(&quot;ES&quot;, &quot;Approach&quot;)

Data_BothApproaches_ALL &lt;- as.data.frame(rbind(Data_trad, Data_phylo))


Plot_AllApproaches1 &lt;-  ggplot() +
  geom_density(data = subset(Data_BothApproaches_ALL, Approach == &quot;Non-phylogenetic&quot;), 
               mapping = aes(x = ES, y = ..density.., fill = Approach),
               alpha = 0.7) +
  geom_density(data = subset(Data_BothApproaches_ALL, Approach == &quot;Phylogenetic&quot;), 
               mapping = aes(x = ES, y = ..density.., fill = Approach),
               alpha = 0.7) +
  scale_colour_manual(values = c(CoffeePalette(20)[5], CoffeePalette(20)[17])) +
  scale_fill_manual(values = c(CoffeePalette(20)[5], CoffeePalette(20)[17])) +
  geom_vline(xintercept = 0, linetype = &quot;longdash&quot;, colour = &quot;black&quot;, size = 0.7) +
  labs(title = &quot;&quot;, y = &quot;Density&quot;, x = &quot;Female Bateman gradient (*r*)&quot;) +
  scale_x_continuous(limits = c(-0.4, 1.1), breaks = c(-0.4, 0, 0.4, 0.8)) +
  scale_y_continuous(limits = c(-3, 11), breaks = c(0, 2, 4, 6, 8, 10)) +
  theme_density(axis.title.x = element_markdown()) +
  theme(legend.spacing = unit(5.0, &quot;in&quot;))

    # jitter
CI &lt;- HPDinterval(mcmc(Model_ALL_phylo$Sol[,&quot;(Intercept)&quot;]))
aggdata_BothApproches &lt;- data.frame(&quot;Approach&quot; = c(&quot;A&quot;, &quot;B&quot;, &quot;C&quot;, &quot;D&quot;, &quot;E&quot;),
                                    &quot;Mean&quot; = c(mean(Data_phylo$ES), 2, 3, 4, 5), 
                                    &quot;l025&quot; = c(CI[1], 2, 3, 4, 5),  
                                    &quot;u975&quot; = c(CI[2], 2, 3, 4, 5))
aggdata_BothApproches$Approach &lt;- as.factor(aggdata_BothApproches$Approach)
Data$Jitter2 &lt;- rep(&quot;A&quot;, nrow(Data))
rm(CI)

Plot_AllApproaches2 &lt;- ggplot() + 
  geom_pointrange(data = aggdata_BothApproches, 
                  mapping = aes(x = Mean, 
                                y = Approach, 
                                xmin = l025, 
                                xmax = u975,
                                colour = Approach),
                  shape = 23,
                  size = 1.5,
                  fatten = 5,
                  stroke = 1.75,
                  fill = &quot;white&quot;) +
  scale_colour_manual(values = c(CoffeePalette(20)[18], &quot;black&quot;, &quot;black&quot;, &quot;black&quot;, &quot;black&quot;)) +
  new_scale_colour() +
  geom_jitter(Data, 
             mapping = aes(x = r_f,
                           y = Jitter2,
                           colour = Jitter2,
                           fill = Jitter2), 
             size = log(1 / Data$r_VAR_f), 
             alpha = 0.4,
             position = position_jitter(height = .35)) +
  scale_fill_manual(values = CoffeePalette(20)[19]) +
  scale_colour_manual(values = CoffeePalette(20)[19]) +
  new_scale_colour() +
  geom_point(data = aggdata_BothApproches, 
                  mapping = aes(x = Mean, 
                                y = Approach, 
                                colour = Approach),
                  shape = 23,
                  size = 6.5,
                  stroke = 1.75,
                  fill = alpha(&quot;white&quot;, .75)) +
  geom_vline(xintercept = 0, linetype = &quot;longdash&quot;, colour = &quot;black&quot;, size = 0.7) +
  scale_colour_manual(values = c(CoffeePalette(20)[18], &quot;black&quot;, &quot;black&quot;, &quot;black&quot;, &quot;black&quot;)) +
  guides(&quot;fill&quot; = &quot;none&quot;, &quot;colour&quot; = &quot;none&quot;) +
  scale_x_continuous(limits = c(-0.4, 1.1), breaks = c(-0.4, 0, 0.4, 0.8)) +
  scale_y_discrete(labels = c(&quot;0&quot;, &quot;1&quot;, &quot;2&quot;, &quot;3&quot;, &quot;10&quot;)) +
  labs(title = &quot;&quot;, y = &quot;Density&quot;, x = &quot;Female Bateman gradient (*r*)&quot;) +
  theme_jitter(axis.title.x = element_markdown())

  # print
grid.newpage()
pushViewport(viewport(layout = grid.layout(nrow = 1, ncol = 1, widths = unit(1, &quot;npc&quot;)))) 
print(Plot_AllApproaches1, vp = viewport(layout.pos.row = 1, layout.pos.col = 1))
print(Plot_AllApproaches2, vp = viewport(layout.pos.row = 1, layout.pos.col = 1))</code></pre>
<p><img src="figure/META_SexSelFem.Rmd/Overall-1.png" width="768" style="display: block; margin: auto;" /> <strong>Figure 1A.</strong> Meta-analytic evidence for sexual selection in females and its relation to the mating system. Global effect size of the Bateman gradient obtained from Generalized Linear Mixed Models (GLMMs) with or without accounting for phylogenetic non-independence (phylogenetic or non-phylogenetic, respectively).</p>
</div>
<div id="moderator-variables" class="section level2">
<h2>Moderator variables</h2>
<p>Secondly, we ran individual GLMMs in which we defined mating success method (copulatory versus genetic), mating success range (with versus without zero-mating success category) or study type (field versus laboratory studies) as a fixed factor to explain inter-study variation in r.<br />
These models account for phylogenetic non-independence and include the phylogeny as a random effect.</p>
<div id="mating-success-method" class="section level3">
<h3>Mating success method</h3>
<pre class="r"><code>MetaData_cMS              &lt;- subset(Data, MS_estimate == &quot;cMS&quot;)
MetaData_gMS              &lt;- subset(Data, MS_estimate == &quot;gMS&quot;)

# prune tree ####
MetaData_cMS_Species_Data &lt;- unique(MetaData_cMS$animal)
Tree_cMS                  &lt;- drop.tip(Tree, Tree$tip.label[-na.omit(match(MetaData_cMS_Species_Data, Tree$tip.label))])

MetaData_gMS_Species_Data &lt;- unique(MetaData_gMS$animal)
Tree_gMS                  &lt;- drop.tip(Tree,  Tree$tip.label[-na.omit(match(MetaData_gMS_Species_Data, Tree$tip.label))])

# predictor model ####
Model_MS_estimate         &lt;- MCMCglmm(r_f~factor(MS_estimate),
                                      random = ~animal + Index + Study_ID,
                                      mev = Data$r_VAR_f,
                                      pedigree = Tree,
                                      data = Data,
                                      prior = pr2, pr = TRUE, verbose = FALSE,
                                      burnin = BURNIN, nitt = NITT, thin = THIN)

# null models ####
Model_cMS                 &lt;- MCMCglmm(r_f~1,
                                      random = ~animal + Index + Study_ID,
                                      mev = MetaData_cMS$r_VAR_f,
                                      pedigree = Tree_cMS,
                                      data = MetaData_cMS,
                                      prior = pr2, pr = TRUE, verbose = FALSE,
                                      burnin = BURNIN, nitt = NITT, thin = THIN)

Model_gMS                 &lt;- MCMCglmm(r_f~1,
                                      random = ~animal + Index + Study_ID,
                                      mev = MetaData_gMS$r_VAR_f,
                                      pedigree = Tree_gMS,
                                      data = MetaData_gMS,
                                      prior = pr2, pr = TRUE, verbose = FALSE,
                                      burnin = BURNIN, nitt = NITT, thin = THIN)


# Raincloud plot ####
  # density
Data_MSest_cMS     &lt;- as.data.frame(Model_cMS$Sol[, &quot;(Intercept)&quot;])
Data_MSest_cMS$MatSys &lt;- as.factor(rep(&quot;cMS&quot;, nrow(Data_MSest_cMS)))
colnames(Data_MSest_cMS) &lt;- c(&quot;ES&quot;, &quot;MSest&quot;)

Data_MSest_gMS     &lt;- as.data.frame(Model_gMS$Sol[, &quot;(Intercept)&quot;])
Data_MSest_gMS$MatSys &lt;- as.factor(rep(&quot;gMS&quot;, nrow(Data_MSest_gMS)))
colnames(Data_MSest_gMS) &lt;- c(&quot;ES&quot;, &quot;MSest&quot;)

Data_MSest_ALL &lt;- as.data.frame(rbind(Data_MSest_cMS, Data_MSest_gMS))


Plot_MSestimate1 &lt;-  ggplot() +
  geom_density(data = Data_MSest_ALL, 
               aes(x = ES, y = ..density.., fill = MSest),
               alpha = 0.7) +
  scale_colour_manual(values = c(TealPalette(20)[9], TealPalette(20)[17])) +
  scale_fill_manual(values = c(TealPalette(20)[9], TealPalette(20)[17])) + 
  geom_vline(xintercept = 0, linetype = &quot;longdash&quot;, colour = &quot;black&quot;, size = 0.7) +
  labs(title = &quot;&quot;, x = &quot;&quot;, y = &quot;Density&quot;) +
  scale_x_continuous(limits = c(-0.4, 1.1), breaks = c(-0.4, 0, 0.4, 0.8)) +
  scale_y_continuous(limits = c(-3.8, 7), breaks = c(0, 2, 4, 6)) +
  theme_density(axis.title.x = element_markdown(),
                axis.title.y = element_text(size = 22))

  # jitter
CI_1 &lt;- HPDinterval(mcmc(Model_cMS$Sol[,&quot;(Intercept)&quot;]))
CI_2 &lt;- HPDinterval(mcmc(Model_gMS$Sol[,&quot;(Intercept)&quot;]))
aggdata_MSest &lt;- data.frame(&quot;Jitter&quot; = c(&quot;B&quot;, &quot;A&quot;, &quot;C&quot;, &quot;D&quot;, &quot;E&quot;, &quot;F&quot;),
                             &quot;MSest&quot; = c(&quot;cMS&quot;, &quot;gMS&quot;, &quot;C&quot;, &quot;D&quot;, &quot;E&quot;, &quot;F&quot;), 
                             &quot;Mean&quot; = c(mean(mcmc(Model_cMS$Sol[, &quot;(Intercept)&quot;])), 
                                        mean(mcmc(Model_gMS$Sol[, &quot;(Intercept)&quot;])), 
                                        5, 5, 5, 5), 
                             &quot;l025&quot; = c(CI_1[1],
                                        CI_2[1], 
                                        5, 5, 5, 5), 
                             &quot;u975&quot; = c(CI_1[2],
                                        CI_2[2],
                                        5, 5, 5, 5))
aggdata_MSest$Jitter &lt;- as.factor(aggdata_MSest$Jitter)
aggdata_MSest$MSest &lt;- as.factor(aggdata_MSest$MSest)
Data$Jitter2 &lt;- as.factor(Data$MS_estimate)
levels(Data$Jitter2) &lt;- c(&quot;B&quot;, &quot;A&quot;)
rm(CI_1, CI_2)


Plot_MSestimate2 &lt;- ggplot() + 
  geom_pointrange(data = aggdata_MSest, 
                  mapping = aes(x = Mean, 
                                y = Jitter, 
                                xmin = l025, 
                                xmax = u975, 
                                colour = MSest),
                  shape = 23,
                  size = 1.5,
                  fatten = 5,
                  stroke = 1.75,
                  fill = &quot;white&quot;,
                  position = position_dodge(width = .7)) +
  guides(&quot;colour&quot; = &quot;none&quot;) +
  scale_colour_manual(values = c(&quot;black&quot;, TealPalette(20)[11], &quot;black&quot;, &quot;black&quot;, &quot;black&quot;, TealPalette(20)[17])) +
  new_scale_colour() +
  geom_jitter(data = Data, 
              mapping = aes(x = r_f,
                            y = Jitter2,
                            fill = MS_estimate,
                            colour = MS_estimate,
                            alpha = MS_estimate), 
              position = position_jitter(height = 0.35), 
              size = log(1 / Data$r_VAR_f)) +
  scale_alpha_manual(values = c(0.5, 0.4)) +
  scale_colour_manual(values = c(TealPalette(20)[11], TealPalette(20)[17])) +
  scale_fill_manual(values = c(TealPalette(20)[11], TealPalette(20)[17])) + 
  new_scale_colour() +
  geom_point(data = aggdata_MSest, 
             mapping = aes(x = Mean, 
                           y = Jitter, 
                           colour = MSest),
             shape = 23,
             size = 6.5,
             stroke = 1.75,
             fill = alpha(&quot;white&quot;, .75),
             position = position_dodge(width = .8)) +
  guides(&quot;colour&quot; = &quot;none&quot;) +
  geom_vline(xintercept = 0, linetype = &quot;longdash&quot;, colour = &quot;black&quot;, size = 0.7) +
  scale_colour_manual(values = c(&quot;black&quot;, TealPalette(20)[11], &quot;black&quot;, &quot;black&quot;, &quot;black&quot;, TealPalette(20)[17])) +
  scale_x_continuous(limits = c(-0.4, 1.1), breaks = c(-0.4, 0, 0.4, 0.8)) +
  scale_y_discrete(labels = c(&quot;0&quot;, &quot;1&quot;, &quot;2&quot;, &quot;3&quot;, &quot;4&quot;, &quot;5&quot;)) +
  labs(title = &quot;&quot;, x = &quot;&quot;, y = &quot;Density&quot;) +
  theme_jitter(axis.title.x = element_markdown(),
               axis.title.y = element_text(size = 22))</code></pre>
</div>
<div id="zero-mating-success---mating-success-range" class="section level3">
<h3>Zero mating success - Mating success range</h3>
<pre class="r"><code>MetaData_with_0MS                 &lt;- subset(Data, Zero_f == &quot;1&quot;)
MetaData_without_0MS              &lt;- subset(Data, Zero_f == &quot;0&quot;)


# prune tree ####
MetaData_with_0MS_Species_Data    &lt;- unique(MetaData_with_0MS$animal)
Tree_with_0MS                     &lt;- drop.tip(Tree, Tree$tip.label[-na.omit(match(MetaData_with_0MS_Species_Data, Tree$tip.label))])

MetaData_without_0MS_Species_Data &lt;- unique(MetaData_without_0MS$animal)
Tree_without_0MS                  &lt;- drop.tip(Tree, Tree$tip.label[-na.omit(match(MetaData_without_0MS_Species_Data, Tree$tip.label))])

# predictor model ####
Model_zeroMS                      &lt;- MCMCglmm(r_f~factor(Zero_f),
                                              random = ~animal + Index + Study_ID,
                                              mev = Data$r_VAR_f,
                                              pedigree = Tree,
                                              data = Data,
                                              prior = pr2, pr = TRUE, verbose = FALSE,
                                              burnin = BURNIN, nitt = NITT, thin = THIN)

# null models ####
Model_with_0MS                   &lt;- MCMCglmm(r_f~1,
                                             random = ~animal + Index + Study_ID,
                                             mev = MetaData_with_0MS$r_VAR_f,
                                             pedigree = Tree_with_0MS,
                                             data = MetaData_with_0MS,
                                             prior = pr2, pr = TRUE, verbose = FALSE,
                                             burnin = BURNIN, nitt = NITT, thin = THIN)

Model_without_0MS                &lt;- MCMCglmm(r_f~1,
                                             random = ~animal + Index + Study_ID,
                                             mev = MetaData_without_0MS$r_VAR_f,
                                             pedigree = Tree_without_0MS,
                                             data = MetaData_without_0MS,
                                             prior = pr2, pr = TRUE, verbose = FALSE,
                                             burnin = BURNIN, nitt = NITT, thin = THIN)


# Raincloud plot ####
  # density
Data_ZeroMS_with &lt;- as.data.frame(Model_with_0MS$Sol[, &quot;(Intercept)&quot;])
Data_ZeroMS_with$ZeroMS &lt;- as.factor(rep(&quot;With Zero mating success&quot;, nrow(Data_ZeroMS_with)))
colnames(Data_ZeroMS_with) &lt;- c(&quot;ES&quot;, &quot;ZeroMS&quot;)

Data_ZeroMS_without &lt;- as.data.frame(Model_without_0MS$Sol[, &quot;(Intercept)&quot;])
Data_ZeroMS_without$ZeroMS &lt;- as.factor(rep(&quot;Without Zero mating success&quot;, nrow(Data_ZeroMS_without)))
colnames(Data_ZeroMS_without) &lt;- c(&quot;ES&quot;, &quot;ZeroMS&quot;)

Data_ZeroMS_ALL &lt;- as.data.frame(rbind(Data_ZeroMS_with, Data_ZeroMS_without))


Plot_ZeroMS1 &lt;-  ggplot() +
  geom_density(data = subset(Data_ZeroMS_ALL, Data_ZeroMS_ALL$ZeroMS == &quot;Without Zero mating success&quot;), 
               aes(x = ES, y = ..density.., fill = &quot;With zero MS&quot;),
               alpha = 0.7) +
  geom_density(data = subset(Data_ZeroMS_ALL, Data_ZeroMS_ALL$ZeroMS == &quot;With Zero mating success&quot;), 
               aes(x = ES, y = ..density.., fill = &quot;Without zero MS&quot;),
               alpha = 0.7) +
  scale_colour_manual(values = c(TealPalette(20)[9], TealPalette(20)[17])) +
  scale_fill_manual(labels = c(&quot;Without zero MS&quot;, &quot;With zero MS&quot;), values = c(TealPalette(20)[9], TealPalette(20)[17])) + 
  geom_vline(xintercept = 0, linetype = &quot;longdash&quot;, colour = &quot;black&quot;, size = 0.7) +
  labs(title = &quot;&quot;, x = &quot;&quot;, y = &quot;Density&quot;) +
  scale_x_continuous(limits = c(-0.4, 1.1), breaks = c(-0.4, 0, 0.4, 0.8)) +
  scale_y_continuous(limits = c(-2.8, 5), breaks = c(0, 1, 2, 3, 4, 5)) +
  theme_density(axis.title.x = element_markdown(),
                axis.title.y = element_text(size = 22))


  # jitter
CI_1 &lt;- HPDinterval(mcmc(Model_with_0MS$Sol[,&quot;(Intercept)&quot;]))
CI_2 &lt;- HPDinterval(mcmc(Model_without_0MS$Sol[,&quot;(Intercept)&quot;]))
aggdata_ZeroMS &lt;- data.frame(&quot;Jitter&quot; = c(&quot;A&quot;, &quot;B&quot;, &quot;C&quot;, &quot;D&quot;, &quot;E&quot;, &quot;F&quot;),
                             &quot;ZeroMS&quot; = c(&quot;With Zero mating success&quot;, &quot;Without Zero mating success&quot;, &quot;C&quot;, &quot;D&quot;, &quot;E&quot;, &quot;F&quot;), 
                             &quot;Mean&quot; = c(mean(mcmc(Model_with_0MS$Sol[, &quot;(Intercept)&quot;])), 
                                        mean(mcmc(Model_without_0MS$Sol[, &quot;(Intercept)&quot;])),
                                        5, 5, 5, 5), 
                             &quot;l025&quot; = c(CI_1[1],
                                        CI_2[1], 
                                        5, 5, 5, 5), 
                             &quot;u975&quot; = c(CI_1[2],
                                        CI_2[2],
                                        5, 5, 5, 5))
aggdata_ZeroMS$Jitter &lt;- as.factor(aggdata_ZeroMS$Jitter)
aggdata_ZeroMS$ZeroMS &lt;- as.factor(aggdata_ZeroMS$ZeroMS)
Data$Jitter2 &lt;- as.factor(Data$Zero_f)
levels(Data$Jitter2) &lt;- c(&quot;A&quot;, &quot;B&quot;)
rm(CI_1, CI_2)


Plot_ZeroMS2 &lt;- ggplot() + 
  geom_pointrange(data = aggdata_ZeroMS, 
                  mapping = aes(x = Mean, 
                                y = Jitter, 
                                xmin = l025, 
                                xmax = u975, 
                                colour = ZeroMS),
                  shape = 23,
                  size = 1.5,
                  fatten = 5,
                  stroke = 1.75,
                  fill = &quot;white&quot;,
                  position = position_dodge(width = .8)) +
  guides(&quot;colour&quot; = &quot;none&quot;) +
  scale_colour_manual(values = c(&quot;black&quot;, &quot;black&quot;, &quot;black&quot;, &quot;black&quot;, TealPalette(20)[17], TealPalette(20)[11])) +
  new_scale_colour() +
  geom_jitter(data = Data, 
              mapping = aes(x = r_f,
                            y = Jitter2,
                            fill = factor(Zero_f),
                            colour = factor(Zero_f),
                            alpha = factor(Zero_f)), 
              position = position_jitter(height = 0.35), 
              size = log(1 / Data$r_VAR_f)) +
  scale_alpha_manual(values = c(0.4, 0.5)) +
  scale_colour_manual(values = c(TealPalette(20)[17], TealPalette(20)[11])) +
  scale_fill_manual(values = c(TealPalette(20)[17], TealPalette(20)[11])) + 
  new_scale_colour() +
  geom_point(data = aggdata_ZeroMS, 
             mapping = aes(x = Mean, 
                           y = Jitter, 
                           colour = ZeroMS),
             shape = 23,
             size = 6.5,
             stroke = 1.75,
             fill = alpha(&quot;white&quot;, .75),
             position = position_dodge(width = .7)) +
  guides(&quot;colour&quot; = &quot;none&quot;) +
  geom_vline(xintercept = 0, linetype = &quot;longdash&quot;, colour = &quot;black&quot;, size = 0.7) +
  scale_colour_manual(values = c(&quot;black&quot;, &quot;black&quot;, &quot;black&quot;, &quot;black&quot;, TealPalette(20)[17], TealPalette(20)[11])) +
  scale_x_continuous(limits = c(-0.4, 1.1), breaks = c(-0.4, 0, 0.4, 0.8)) +
  scale_y_discrete(labels = c(&quot;0&quot;, &quot;1&quot;, &quot;2&quot;, &quot;3&quot;, &quot;4&quot;, &quot;5&quot;)) +
  labs(title = &quot;&quot;, x = &quot;&quot;, y = &quot;Density&quot;) +
  theme_jitter(axis.title.x = element_markdown(),
               axis.title.y = element_text(size = 22))</code></pre>
</div>
<div id="study-type" class="section level3">
<h3>Study type</h3>
<pre class="r"><code>MetaData_Field              &lt;- subset(Data, StudyType == &quot;Field&quot;)
MetaData_Lab                &lt;- subset(Data, StudyType == &quot;Lab&quot;)


# prune tree ####
MetaData_Field_Species_Data &lt;- unique(MetaData_Field$animal)
Tree_Field                  &lt;- drop.tip(Tree, Tree$tip.label[-na.omit(match(MetaData_Field_Species_Data, Tree$tip.label))])

MetaData_Lab_Species_Data   &lt;- unique(MetaData_Lab$animal)
Tree_Lab                    &lt;- drop.tip(Tree, Tree$tip.label[-na.omit(match(MetaData_Lab_Species_Data, Tree$tip.label))])

# predictor model ####
Model_StudyType             &lt;- MCMCglmm(r_f~StudyType,
                                        random = ~animal + Index + Study_ID,
                                        mev = Data$r_VAR_f,
                                        pedigree = Tree,
                                        data = Data,
                                        prior = pr2, pr = TRUE, verbose = FALSE,
                                        burnin = BURNIN, nitt = NITT, thin = THIN)

# null models ####
Model_Field                 &lt;- MCMCglmm(r_f~1,
                                        random = ~animal + Index + Study_ID,
                                        mev = MetaData_Field$r_VAR_f,
                                        pedigree = Tree_Field,
                                        data = MetaData_Field,
                                        prior = pr2, pr = TRUE, verbose = FALSE,
                                        burnin = BURNIN, nitt = NITT, thin = THIN)

Model_Lab                  &lt;- MCMCglmm(r_f~1,
                                       random = ~animal + Index + Study_ID,
                                       mev = MetaData_Lab$r_VAR_f,
                                       pedigree = Tree_Lab,
                                       data = MetaData_Lab,
                                       prior = pr2, pr = TRUE, verbose = FALSE,
                                       burnin = BURNIN, nitt = NITT, thin = THIN)


# Raincloud plot ####
  # density
Data_StudyType_field     &lt;- as.data.frame(Model_Field$Sol[, &quot;(Intercept)&quot;])
Data_StudyType_field$StudyType &lt;- as.factor(rep(&quot;Field&quot;, nrow(Data_StudyType_field)))
colnames(Data_StudyType_field) &lt;- c(&quot;ES&quot;, &quot;StudyType&quot;)

Data_StudyType_lab     &lt;- as.data.frame(Model_Lab$Sol[, &quot;(Intercept)&quot;])
Data_StudyType_lab$StudyType &lt;- as.factor(rep(&quot;Lab&quot;, nrow(Data_StudyType_lab)))
colnames(Data_StudyType_lab) &lt;- c(&quot;ES&quot;, &quot;StudyType&quot;)

Data_StudyType_ALL &lt;- as.data.frame(rbind(Data_StudyType_field, Data_StudyType_lab))


Plot_StudyType1 &lt;-  ggplot() +
  geom_density(data = Data_StudyType_ALL, 
               aes(x = ES, y = ..density.., fill = StudyType),
               alpha = 0.7) +
  scale_colour_manual(values = c(TealPalette(20)[9], TealPalette(20)[17])) +
  scale_fill_manual(values = c(TealPalette(20)[9], TealPalette(20)[17])) + 
  geom_vline(xintercept = 0, linetype = &quot;longdash&quot;, colour = &quot;black&quot;, size = 0.7) +
  labs(title = &quot;&quot;, x = &quot;Female Bateman gradient (*r*)&quot;, y = &quot;Density&quot;) +
  scale_x_continuous(limits = c(-0.4, 1.1), breaks = c(-0.4, 0, 0.4, 0.8)) +
  scale_y_continuous(limits = c(-2.8, 5), breaks = c(0, 1, 2, 3, 4, 5)) +
  theme_density(axis.title.x = element_markdown(),
                axis.title.y = element_text(size = 22))


  # jitter
CI_1 &lt;- HPDinterval(mcmc(Model_Field$Sol[,&quot;(Intercept)&quot;]))
CI_2 &lt;- HPDinterval(mcmc(Model_Lab$Sol[,&quot;(Intercept)&quot;]))
aggdata_StudyType &lt;- data.frame(&quot;Jitter&quot; = c(&quot;B&quot;, &quot;A&quot;, &quot;C&quot;, &quot;D&quot;, &quot;E&quot;, &quot;F&quot;),
                                &quot;StudyType&quot; = c(&quot;Field&quot;, &quot;Lab&quot;, &quot;C&quot;, &quot;D&quot;, &quot;E&quot;, &quot;F&quot;), 
                                &quot;Mean&quot; = c(mean(mcmc(Model_Field$Sol[, &quot;(Intercept)&quot;])), 
                                           mean(mcmc(Model_Lab$Sol[, &quot;(Intercept)&quot;])), 
                                           5, 5, 5, 5), 
                                &quot;l025&quot; = c(CI_1[1],
                                           CI_2[1], 
                                           5, 5, 5, 5), 
                                &quot;u975&quot; = c(CI_1[2],
                                           CI_2[2],
                                           5, 5, 5, 5))
aggdata_StudyType$Jitter &lt;- as.factor(aggdata_StudyType$Jitter)
aggdata_StudyType$StudyType &lt;- as.factor(aggdata_StudyType$StudyType)
Data$Jitter2 &lt;- as.factor(Data$StudyType)
levels(Data$Jitter2) &lt;- c(&quot;B&quot;, &quot;A&quot;)
rm(CI_1, CI_2)


Plot_StudyType2 &lt;- ggplot() + 
  geom_pointrange(data = aggdata_StudyType, 
                  mapping = aes(x = Mean, 
                                y = Jitter, 
                                xmin = l025, 
                                xmax = u975, 
                                colour = StudyType),
                  shape = 23,
                  size = 1.5,
                  fatten = 5,
                  stroke = 1.75,
                  fill = &quot;white&quot;,
                  position = position_dodge(width = .8)) +
  guides(&quot;colour&quot; = &quot;none&quot;) +
  scale_colour_manual(values = c(&quot;black&quot;, &quot;black&quot;, &quot;black&quot;, &quot;black&quot;, TealPalette(20)[11], TealPalette(20)[17])) +
  new_scale_colour() +
  geom_jitter(data = Data, 
              mapping = aes(x = r_f,
                            y = Jitter2,
                            fill = StudyType,
                            colour = StudyType,
                            alpha = StudyType), 
              position = position_jitter(height = 0.35), 
              size = log(1 / Data$r_VAR_f)) +
  scale_alpha_manual(values = c(0.5, 0.4)) +
  scale_colour_manual(values = c(TealPalette(20)[11], TealPalette(20)[17])) +
  scale_fill_manual(values = c(TealPalette(20)[11], TealPalette(20)[17])) + 
  new_scale_colour() +
  geom_point(data = aggdata_StudyType, 
             mapping = aes(x = Mean, 
                           y = Jitter, 
                           colour = StudyType),
             shape = 23,
             size = 6.5,
             stroke = 1.75,
             fill = alpha(&quot;white&quot;, .75),
             position = position_dodge(width = .7)) +
  guides(&quot;colour&quot; = &quot;none&quot;) +
  geom_vline(xintercept = 0, linetype = &quot;longdash&quot;, colour = &quot;black&quot;, size = 0.7) +
  scale_colour_manual(values = c(&quot;black&quot;, &quot;black&quot;, &quot;black&quot;, &quot;black&quot;, TealPalette(20)[11], TealPalette(20)[17])) +
  scale_x_continuous(limits = c(-0.4, 1.1), breaks = c(-0.4, 0, 0.4, 0.8)) +
  scale_y_discrete(labels = c(&quot;0&quot;, &quot;1&quot;, &quot;2&quot;, &quot;3&quot;, &quot;4&quot;, &quot;5&quot;)) +
  labs(title = &quot;&quot;, x = &quot;Female Bateman gradient (*r*)&quot;, y = &quot;Density&quot;) +
  theme_jitter(axis.title.x = element_markdown(),
               axis.title.y = element_text(size = 22))</code></pre>
<pre class="r"><code>Plot_Density2 &lt;- plot_grid(Plot_MSestimate1, Plot_ZeroMS1, Plot_StudyType1,
                           labels = &quot;AUTO&quot;,
                           label_size = 18,
                           hjust = 0, 
                           vjust = 1, 
                           align = &quot;hv&quot;,
                           ncol = 1,
                           nrow = 3)
Plot_Jitter2 &lt;- plot_grid(Plot_MSestimate2, Plot_ZeroMS2, Plot_StudyType2,
                          labels = &quot;AUTO&quot;,
                          label_size = 18,
                          hjust = 0, 
                          vjust = 1, 
                          align = &quot;hv&quot;,
                          ncol = 1,
                          nrow = 3)

grid.newpage()
pushViewport(viewport(layout = grid.layout(nrow = 1, ncol = 1, widths = unit(1, &quot;npc&quot;)))) 
print(Plot_Density2, vp = viewport(layout.pos.row = 1, layout.pos.col = 1))
print(Plot_Jitter2, vp = viewport(layout.pos.row = 1, layout.pos.col = 1))</code></pre>
<p><img src="figure/META_SexSelFem.Rmd/Plots-1.png" width="768" style="display: block; margin: auto;" /> <strong>Figure S3.</strong> Methodological predictors of female Bateman gradients. Raincloud charts showing effects of mating success method (cMS: copulatory mating success, gMS: genetic mating success), mating success range (with or without zero mating success (MS) category) and study type (field versus laboratory studies) on female Bateman gradients (for statistical analysis see Table 2 and S2).</p>
</div>
<div id="mating-system" class="section level3">
<h3>Mating system</h3>
<p>We classified the mating system of each sampled species based on estimates of polyandry, which we defined as the proportion of reproducing females that have more than one mating partner. We then used these estimates to define the mating system as either monandrous or polyandrous, depending on whether its value was lower or higher than 0.5, respectively.</p>
<pre class="r"><code>MetaData_polygamous              &lt;- subset(Data, MatingSystem == &quot;polyandrous&quot;)
MetaData_monogamous              &lt;- subset(Data, MatingSystem == &quot;monogamous&quot;)

# prune tree ####
MetaData_polygamous_Species_Data &lt;- unique(MetaData_polygamous$animal)
Tree_polygamous                  &lt;- drop.tip(Tree, Tree$tip.label[-na.omit(match(MetaData_polygamous_Species_Data, Tree$tip.label))])

MetaData_monogamous_Species_Data &lt;- unique(MetaData_monogamous$animal)
Tree_monogamous                  &lt;- drop.tip(Tree, Tree$tip.label[-na.omit(match(MetaData_monogamous_Species_Data, Tree$tip.label))])


# predictor model ####
Model_MatingSystem               &lt;- MCMCglmm(r_f~MatingSystem,
                                             random = ~animal + Index + Study_ID,
                                             mev = Data$r_VAR_f,
                                             pedigree = Tree,
                                             data = Data,
                                             prior = pr2, pr = TRUE, verbose = FALSE,
                                             burnin = BURNIN, nitt = NITT, thin = THIN)

# null models ####
Model_monogamous                 &lt;- MCMCglmm(r_f~1,
                                             random = ~animal + Index + Study_ID,
                                             mev = MetaData_monogamous$r_VAR_f,
                                             pedigree = Tree_monogamous,
                                             data = MetaData_monogamous,
                                             prior = pr2, pr = TRUE, verbose = FALSE,
                                             burnin = BURNIN, nitt = NITT, thin = THIN)

Model_polygamous                 &lt;- MCMCglmm(r_f~1,
                                             random = ~animal + Index + Study_ID,
                                             mev = MetaData_polygamous$r_VAR_f,
                                             pedigree = Tree_polygamous,
                                             data = MetaData_polygamous,
                                             prior = pr2, pr = TRUE, verbose = FALSE,
                                             burnin = BURNIN, nitt = NITT, thin = THIN)


# Raincloud plot ####
  # density
Data_MatSys_Monogamy     &lt;- as.data.frame(Model_monogamous$Sol[, &quot;(Intercept)&quot;])
Data_MatSys_Monogamy$MatSys &lt;- as.factor(rep(&quot;monandrous&quot;, nrow(Data_MatSys_Monogamy)))
colnames(Data_MatSys_Monogamy) &lt;- c(&quot;ES&quot;, &quot;MatSys&quot;)

Data_MatSys_Polygamy     &lt;- as.data.frame(Model_polygamous$Sol[, &quot;(Intercept)&quot;])
Data_MatSys_Polygamy$MatSys &lt;- as.factor(rep(&quot;Polyandrous&quot;, nrow(Data_MatSys_Polygamy)))
colnames(Data_MatSys_Polygamy) &lt;- c(&quot;ES&quot;, &quot;MatSys&quot;)

Data_MatingSystem_ALL &lt;- as.data.frame(rbind(Data_MatSys_Monogamy, Data_MatSys_Polygamy))


Plot_MatingSystem1 &lt;-  ggplot() +
  geom_density(data = Data_MatingSystem_ALL, 
               aes(x = ES, y = ..density.., fill = MatSys),
               alpha = 0.7) +
  scale_colour_manual(values = c(TealPalette(20)[9], TealPalette(20)[17])) +
  scale_fill_manual(values = c(TealPalette(20)[9], TealPalette(20)[17])) + 
  geom_vline(xintercept = 0, linetype = &quot;longdash&quot;, colour = &quot;black&quot;, size = 0.7) +
  labs(title = &quot;&quot;, x = &quot;Female Bateman gradient (*r*)&quot;, y = &quot;Density&quot;) +
  scale_x_continuous(limits = c(-0.4, 1.1), breaks = c(-0.4, 0, 0.4, 0.8)) +
  scale_y_continuous(limits = c(-2.5, 5), breaks = c(0, 1, 2, 3, 4, 5)) +
  theme_density(axis.title.x = element_markdown())


  # jitter
CI_1 &lt;- HPDinterval(mcmc(Model_monogamous$Sol[,&quot;(Intercept)&quot;]))
CI_2 &lt;- HPDinterval(mcmc(Model_polygamous$Sol[,&quot;(Intercept)&quot;]))
aggdata_MatSys &lt;- data.frame(&quot;Jitter&quot; = c(&quot;A&quot;, &quot;B&quot;, &quot;C&quot;, &quot;D&quot;, &quot;E&quot;, &quot;F&quot;),
                             &quot;MatSys&quot; = c(&quot;Polyandrous&quot;, &quot;monandrous&quot;, &quot;C&quot;, &quot;D&quot;, &quot;E&quot;, &quot;F&quot;), 
                             &quot;Mean&quot; = c(mean(mcmc(Model_polygamous$Sol[, &quot;(Intercept)&quot;])), 
                                        mean(mcmc(Model_monogamous$Sol[, &quot;(Intercept)&quot;])), 
                                        5, 5, 5, 5), 
                             &quot;l025&quot; = c(CI_2[1],
                                        CI_1[1], 
                                        5, 5, 5, 5), 
                             &quot;u975&quot; = c(CI_2[2],
                                        CI_1[2],
                                        5, 5, 5, 5))
aggdata_MatSys$Jitter &lt;- as.factor(aggdata_MatSys$Jitter)
aggdata_MatSys$MatSys &lt;- as.factor(aggdata_MatSys$MatSys)
Data$Jitter2 &lt;- as.factor(Data$MatingSystem)
levels(Data$Jitter2) &lt;- c(&quot;B&quot;, &quot;A&quot;)
rm(CI_1, CI_2)


Plot_MatingSystem2 &lt;- ggplot() + 
  geom_pointrange(data = aggdata_MatSys, 
                  mapping = aes(x = Mean, 
                                y = Jitter, 
                                xmin = l025, 
                                xmax = u975, 
                                colour = MatSys),
                  shape = 23,
                  size = 1.5,
                  fatten = 5,
                  stroke = 1.75,
                  fill = &quot;white&quot;,
                  position = position_dodge(width = .8)) +
  scale_colour_manual(values = c(&quot;black&quot;, &quot;black&quot;, &quot;black&quot;, &quot;black&quot;, TealPalette(20)[11], TealPalette(20)[17])) +
  new_scale_colour() +
  geom_jitter(data = Data, 
              mapping = aes(x = r_f,
                            y = Jitter2,
                            fill = MatingSystem,
                            colour = MatingSystem,
                            alpha = MatingSystem), 
              position = position_jitter(height = 0.35), 
              size = log(1 / Data$r_VAR_f)) +
  scale_colour_manual(values = c(TealPalette(20)[11], TealPalette(20)[17])) +
  scale_fill_manual(values = c(TealPalette(20)[11], TealPalette(20)[17])) + 
  scale_alpha_manual(values = c(0.5, 0.4)) +
  new_scale_colour() +
  geom_point(data = aggdata_MatSys, 
                  mapping = aes(x = Mean, 
                                y = Jitter, 
                                colour = MatSys),
                  shape = 23,
                  size = 6.5,
                  stroke = 1.75,
                  fill = alpha(&quot;white&quot;, .75),
                  position = position_dodge(width = .8)) +
  geom_vline(xintercept = 0, linetype = &quot;longdash&quot;, colour = &quot;black&quot;, size = 0.7) +
  scale_colour_manual(values = c(&quot;black&quot;, &quot;black&quot;, &quot;black&quot;, &quot;black&quot;, TealPalette(20)[11], TealPalette(20)[17])) +
  scale_x_continuous(limits = c(-0.4, 1.1), breaks = c(-0.4, 0, 0.4, 0.8)) +
  scale_y_discrete(labels = c(&quot;0&quot;, &quot;1&quot;, &quot;2&quot;, &quot;3&quot;, &quot;4&quot;, &quot;5&quot;)) +
  labs(title = &quot;&quot;, x = &quot;Female Bateman gradient (*r*)&quot;, y = &quot;Density&quot;) +
  theme_jitter(axis.title.x = element_markdown())

  # print
grid.newpage()
pushViewport(viewport(layout = grid.layout(nrow = 1, ncol = 1, widths = unit(1, &quot;npc&quot;)))) 
print(Plot_MatingSystem1, vp = viewport(layout.pos.row = 1, layout.pos.col = 1))
print(Plot_MatingSystem2, vp = viewport(layout.pos.row = 1, layout.pos.col = 1))</code></pre>
<p><img src="figure/META_SexSelFem.Rmd/MatingSystem-1.png" width="768" style="display: block; margin: auto;" /> <strong>Figure 1B.</strong> Meta-analytic evidence for sexual selection in females and its relation to the mating system. Contrast in sexual selection in females between monandrous and polyandrous species. Raincloud charts show posterior distributions, global effect size with 95% Highest Posterior Density intervals (diamonds and error bars) and raw effect sizes (filled circles) of female Bateman gradient.</p>
<div id="polyandry" class="section level4">
<h4>Polyandry</h4>
<p>We also ran GLMMs with the proportion of multiply mated females (i. e., a continuous variable) as a fixed effect.<br />
The studentized Breusch-Pagan test informs on whether this variable is significantly heteroscedastic.</p>
<pre class="r"><code># predictor model ####
Model_Polyandry                  &lt;- MCMCglmm(r_f ~ Polyandry,
                                             random = ~animal + Index + Study_ID,
                                             mev = Data$r_VAR_f,
                                             pedigree = Tree,
                                             data = Data,
                                             prior = pr2, pr = TRUE, verbose = FALSE,
                                             burnin = BURNIN, nitt = NITT, thin = THIN)

## The Breusch-Pagan Test ####
LM_Polyandry_r_f      &lt;- lm(r_f ~ Polyandry, data = Data)

intercept &lt;- summary(Model_Polyandry)$solutions[1, 1]
slope     &lt;- summary(Model_Polyandry)$solutions[2, 1]
slope_l95 &lt;- summary(Model_Polyandry)$solutions[2, 2]
slope_u95 &lt;- summary(Model_Polyandry)$solutions[2, 3]
pointsize_r_f &lt;- log(1/Data$r_VAR_f)

Data$Polyandry_residuals &lt;- LM_Polyandry_r_f$residuals

LM_BreuschPagan_Polyandry &lt;- lm(Polyandry_residuals^2 ~ Polyandry, data = Data)
summary(LM_BreuschPagan_Polyandry)</code></pre>
<pre><code>
Call:
lm(formula = Polyandry_residuals^2 ~ Polyandry, data = Data)

Residuals:
     Min       1Q   Median       3Q      Max 
-0.14365 -0.07949 -0.02605  0.04979  0.42561 

Coefficients:
            Estimate Std. Error t value Pr(&gt;|t|)   
(Intercept)  0.02614    0.02578   1.014  0.31286   
Polyandry    0.12070    0.03979   3.034  0.00302 **
---
Signif. codes:  0 &#39;***&#39; 0.001 &#39;**&#39; 0.01 &#39;*&#39; 0.05 &#39;.&#39; 0.1 &#39; &#39; 1

Residual standard error: 0.1112 on 109 degrees of freedom
Multiple R-squared:  0.07785,   Adjusted R-squared:  0.06939 
F-statistic: 9.203 on 1 and 109 DF,  p-value: 0.003021</code></pre>
<pre class="r"><code># plot ####

Plot_r_f_Polyandry &lt;- ggplot(Data, aes(x = Polyandry, y = r_f)) +
  geom_point(shape = 19, size = pointsize_r_f, alpha = 0.6, colour = &quot;royalblue&quot;) +
  labs(title = &quot;&quot;, x = &quot;Polyandry&quot;, y = &quot;Bateman gradient (*r*)&quot;) +
  scale_x_continuous(limits = c(-0.1, 1.1),
                     breaks = c(0, 0.25, 0.5, 0.75, 1),
                     expand = c(0, 0)) +
  scale_y_continuous(limits = c(-0.5, 1.2),
                     breaks = c(-0.5, 0, 0.5, 1),
                     expand = c(0, 0)) +
  theme_paper(axis.title.y = element_markdown()) +
  geom_abline(aes(intercept = intercept, slope = slope), colour = &quot;black&quot;)
Plot_r_f_Polyandry</code></pre>
<p><img src="figure/META_SexSelFem.Rmd/Polyandry-1.png" width="672" style="display: block; margin: auto;" /></p>
</div>
</div>
<div id="publication-year" class="section level3">
<h3>Publication year</h3>
<p>We tested for the effect of the publication year of the studies on the effect size to investigate the presence or absence of the bandwagon effect.</p>
<pre class="r"><code># predictor model 
Model_Year      &lt;- MCMCglmm(r_f~Year,
                            random = ~animal + Index + Study_ID,
                            mev = Data$r_VAR_f,
                            pedigree = Tree,
                            data = Data,
                            prior = pr2, pr = TRUE, verbose = FALSE,
                            burnin = BURNIN, nitt = NITT, thin = THIN)</code></pre>
</div>
</div>
<div id="heterogeneity" class="section level2">
<h2>Heterogeneity</h2>
<p>We created functions to extract the relevant information from the models and to estimate heterogeneity <em>I²</em>, which we defined as the proportion of variance in effect size that can be attributed to the different levels of random effects <span class="citation">(Higgins and Thompson 2002)</span>. In particular, we decomposed total heterogeneity into the proportional phylogenetic variance (<em>I²_{Phylogeny}</em>), between-study variance (<em>I²_{Study}</em>), and study-specific variance (observation-level random effect; <em>I²_{Observation}</em>) <span class="citation">(Nakagawa and Santos 2012)</span>. For models including predictor variables, we computed the proportion of variance explained by those fixed factors (‘marginal <em>R²</em>’) <span class="citation">(Nakagawa and Schielzeth 2013)</span>.</p>
<pre class="r"><code># Null models ####
NullModel_summary1 &lt;- function(which_data) {
  k &lt;- length(which_data$r_f)  # sample size
  N_sp &lt;- length((aggregate(r_f ~ animal, data = which_data, FUN = sum))$animal)  # number of species
  
  as.data.frame(cbind(k, N_sp))
}

NullModel_summary2 &lt;- function(which_model) {
  # MCMCglmm summary 
  MCMCglmm_summary &lt;- as.data.frame(cbind(
    posterior.mode(which_model$Sol[,&quot;(Intercept)&quot;]),  # posterior mode
    cbind(HPDinterval(which_model$Sol[,&quot;(Intercept)&quot;])), # 95% highest posterior density
    summary(which_model)$solutions[, 5])) #P(MCMC)
  colnames(MCMCglmm_summary) &lt;- c(&quot;Mode&quot;, &quot;Lower HPD&quot;, &quot;Higher HPD&quot;, &quot;pmcmc&quot;)

  # heterogeneity
    ## I2 Phylogenetic signal - Mode and HPD
  H2_null_models  &lt;- which_model$VCV[, &quot;animal&quot;]/(which_model$VCV[, &quot;animal&quot;] +  which_model$VCV[, &quot;Study_ID&quot;] + which_model$VCV[, &quot;Index&quot;] + which_model$VCV[, &quot;units&quot;])
  Mode_H2   &lt;- posterior.mode(H2_null_models)
  HPD_H2    &lt;- HPDinterval(H2_null_models)
  ALL_H2    &lt;- as.data.frame(cbind(Mode_H2, HPD_H2))
  colnames(ALL_H2) &lt;- c(&quot;Mode2&quot;, &quot;Lower HPD2&quot;, &quot;Higher HPD2&quot;)

    ## I2 Study ID variance - Mode and HPD
  Study2_null_models &lt;- which_model$VCV[, &quot;Study_ID&quot;]/(which_model$VCV[, &quot;animal&quot;] +  which_model$VCV[, &quot;Study_ID&quot;] + which_model$VCV[, &quot;Index&quot;] + which_model$VCV[, &quot;units&quot;])
  Mode_Study2 &lt;- posterior.mode(Study2_null_models)
  HPD_Study2 &lt;- HPDinterval(Study2_null_models)
  ALL_Study2 &lt;- as.data.frame(cbind(Mode_Study2, HPD_Study2))
  colnames(ALL_Study2) &lt;- c(&quot;Mode3&quot;, &quot;Lower HPD3&quot;, &quot;Higher HPD3&quot;)

    ## I2 Observation ID variance - Mode and HPD
  Index2_null_models &lt;- which_model$VCV[, &quot;Index&quot;]/(which_model$VCV[, &quot;animal&quot;] +  which_model$VCV[, &quot;Study_ID&quot;] + which_model$VCV[, &quot;Index&quot;] + which_model$VCV[, &quot;units&quot;])
  Mode_Index2 &lt;- posterior.mode(Index2_null_models)
  HPD_Index2 &lt;- HPDinterval(Index2_null_models)
  ALL_Index2 &lt;- as.data.frame(cbind(Mode_Index2, HPD_Index2))
  colnames(ALL_Index2) &lt;- c(&quot;Mode4&quot;, &quot;Lower HPD4&quot;, &quot;Higher HPD4&quot;)
  
  # output
  as.data.frame(cbind(MCMCglmm_summary,
                      ALL_H2,
                      ALL_Study2,
                      ALL_Index2))
}

List_data &lt;- list(Data, 
                  MetaData_cMS, MetaData_gMS, 
                  MetaData_with_0MS, MetaData_without_0MS, 
                  MetaData_Lab, MetaData_Field, 
                  MetaData_monogamous, MetaData_polygamous)

List_Nullmodels &lt;- list(Model_ALL_phylo,
                        Model_cMS, Model_gMS,
                        Model_with_0MS, Model_without_0MS,
                        Model_Lab, Model_Field,
                        Model_monogamous, Model_polygamous)
Model_sum1 &lt;- adply(.data = List_data, .margins = 1, .fun = NullModel_summary1) # result of function as a dataframe: each row is f(x) for a different x from the list
Model_sum2 &lt;- adply(.data = List_Nullmodels, .margins = 1, .fun = NullModel_summary2)


Table1 &lt;- cbind(Model_sum1[,-1], Model_sum2[, -1]) %&gt;%
  mutate_at(c(3:5, 7:15), round, digits = 2)
Table1 &lt;- Table1 %&gt;%
  mutate_at(6, round, digits = 3)

# MCMC model summary - traditional approach
MCMCglmm_Model_ALL_traditional &lt;- as.data.frame(cbind(
  posterior.mode(Model_ALL_traditional$Sol[,&quot;(Intercept)&quot;]), 
  cbind(HPDinterval(Model_ALL_traditional$Sol[,&quot;(Intercept)&quot;])),
  summary(Model_ALL_traditional)$solutions[, 5]))
colnames(MCMCglmm_Model_ALL_traditional) &lt;- c(&quot;Mode&quot;, &quot;Lower HPD&quot;, &quot;Higher HPD&quot;, &quot;pmcmc&quot;)
MCMCglmm_Model_ALL_traditional_sum &lt;- cbind(k &lt;- length(Data$r_f),
                                            N_sp &lt;- length((aggregate(r_f ~ animal, data = Data, FUN = sum))$animal),
                                            MCMCglmm_Model_ALL_traditional,
                                            t(rep(c(&quot;--&quot;, &quot;&quot;, &quot;&quot;), times = 3)))
colnames(MCMCglmm_Model_ALL_traditional_sum) &lt;- c(&quot;k&quot;, &quot;N_sp&quot;, &quot;Mode&quot;, &quot;Lower HPD&quot;, &quot;Higher HPD&quot;, &quot;pmcmc&quot;, &quot;Mode2&quot;, &quot;Lower HPD2&quot;, &quot;Higher HPD2&quot;, &quot;Mode3&quot;, &quot;Lower HPD3&quot;, &quot;Higher HPD3&quot;, &quot;Mode4&quot;, &quot;Lower HPD4&quot;, &quot;Higher HPD4&quot;)
MCMCglmm_Model_ALL_traditional_sum[, 3:5] &lt;- round(MCMCglmm_Model_ALL_traditional_sum[, 3:5], digits = 2)
MCMCglmm_Model_ALL_traditional_sum[, 6] &lt;- round(MCMCglmm_Model_ALL_traditional_sum[, 6], digits = 3)

# Predictor models ####
PredictorModel_summary &lt;- function(which_model) {
  # MCMCglmm summary
  MCMCglmm_summary &lt;- as.data.frame(cbind(
    posterior.mode(which_model$Sol[, 2]),  # posterior mode
    cbind(HPDinterval(which_model$Sol[, 2])), # 95% highest posterior density
    summary(which_model)$solutions[2, 5])) #P(MCMC)
  colnames(MCMCglmm_summary) &lt;- c(&quot;Mode&quot;, &quot;Lower HPD&quot;, &quot;Higher HPD&quot;, &quot;pmcmc&quot;)
  
  
  # Explained variance
  mPred_Model &lt;- mean(which_model$Sol[, 2]) * which_model$X[, 2]
  mVar_mPred_Model &lt;- var(mPred_Model)

  R2_Pred_models &lt;- mVar_mPred_Model/(mVar_mPred_Model + which_model$VCV[, &quot;animal&quot;] + which_model$VCV[, &quot;Study_ID&quot;] + which_model$VCV[, &quot;Index&quot;] + which_model$VCV[, &quot;units&quot;])
  Mode_R2 &lt;- posterior.mode(R2_Pred_models)
  HPD_R2  &lt;- HPDinterval(R2_Pred_models)
  ALL_R2    &lt;- as.data.frame(cbind(Mode_R2, HPD_R2))
  colnames(ALL_R2) &lt;- c(&quot;R2 Mode&quot;, &quot;R2 Lower HPD&quot;, &quot;R2 Higher HPD&quot;)
  
  as.data.frame(cbind(MCMCglmm_summary, ALL_R2))
}

List_Predmodels &lt;- list(Model_MS_estimate,
                        Model_zeroMS,
                        Model_StudyType,
                        Model_Year,
                        Model_MatingSystem,
                        Model_Polyandry)
Table2 &lt;- adply(.data = List_Predmodels, .margins = 1, .fun = PredictorModel_summary) %&gt;%
  select(2:8) %&gt;%
  mutate_at(c(1:3, 5:7), round, digits = 2)
Table2 &lt;- Table2 %&gt;%
  mutate_at(4, round, digits = 3)</code></pre>
</div>
<div id="models-summary" class="section level2">
<h2>Models summary</h2>
<p><strong>Table 1. Global tests of sexual selection in females.</strong> Results of intercept-only phylogenetically controlled General Linear-Mixed Effects Models are shown for the entire dataset (global model) and subsets with respect to mating success method (copulatory versus genetic), mating success range (including versus excluding zero mating success category), study type (laboratory versus field studies) and mating system (monandrous versus polygamous species). Table shows number of effect sizes (k), number of species (N), effect size (r), and heterogeneity I2 arising from phylogenetic affinities, between-study variation, and between-observation variation. Model estimates are shown as posterior modes with 95% Highest Posterior Density (HPD) intervals in brackets.</p>
<pre class="r"><code>for (i in c(4, 8, 11, 14)){
  Table1[, i] &lt;- paste0(&quot;(&quot;, Table1[, i], &quot;, &quot;)}
for (i in c(5, 9, 12, 15)){
  Table1[, i] &lt;- paste0(Table1[, i], &quot;)&quot;)}

MCMCglmm_Model_ALL_traditional_sum$`Lower HPD` &lt;- paste0(&quot;(&quot;, MCMCglmm_Model_ALL_traditional_sum$`Lower HPD`, &quot;, &quot;)
MCMCglmm_Model_ALL_traditional_sum$`Higher HPD` &lt;- paste0(MCMCglmm_Model_ALL_traditional_sum$`Higher HPD`, &quot;)&quot;)

# All null models summary
Table1 &lt;- rbind(MCMCglmm_Model_ALL_traditional_sum, Table1)
colnames(Table1) &lt;- c(&quot;&quot;, &quot;&quot;, &quot;r&quot;, &quot;&quot;, &quot;&quot;, &quot;P~MCMC~&quot;, 
                      &quot;I²~Phylogeny~&quot;, &quot;&quot;, &quot;&quot;, 
                      &quot;I²~Study~&quot;, &quot;&quot;, &quot;&quot;,
                      &quot;I²~Observation~&quot;, &quot;&quot;, &quot;&quot;)
rownames(Table1) &lt;- c(&quot;Global model (non-phylogenetic)&quot;,
                      &quot;Global model (phylogenetic)&quot;,
                      &quot;Copulatory mating success&quot;,
                      &quot;Genetic mating success&quot;,
                      &quot;Including zero mating success&quot;,
                      &quot;Excluding zero mating success&quot;,
                      &quot;Laboratory studies&quot;,
                      &quot;Field studies&quot;,
                      &quot;monandrous species&quot;,
                      &quot;Polyandrous species&quot;)
Table1[1, 6] &lt;- &quot;&lt;0.001&quot;

Table1 %&gt;%
kable(align = c(rep(&quot;c&quot;, 3), &quot;r&quot;, &quot;l&quot;, &quot;r&quot;, rep(c(&quot;c&quot;, &quot;r&quot;, &quot;l&quot;), 3))) %&gt;% 
  kable_paper(full_width = TRUE) %&gt;%
  add_header_above(c(&quot;Model&quot; = 1, &quot;k&quot; = 1, &quot;N~Species~&quot; = 1, &quot;Effect size&quot; = 4, &quot;Heterogeneity&quot; = 9)) %&gt;%
  row_spec(0, italic = TRUE, font_size = 14)</code></pre>
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="1">
<div style="border-bottom: 1px solid #00000020; padding-bottom: 5px; ">
Model
</div>
</th>
<th style="padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="1">
<div style="border-bottom: 1px solid #00000020; padding-bottom: 5px; ">
k
</div>
</th>
<th style="padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="1">
<div style="border-bottom: 1px solid #00000020; padding-bottom: 5px; ">
N<sub>Species</sub>
</div>
</th>
<th style="padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="4">
<div style="border-bottom: 1px solid #00000020; padding-bottom: 5px; ">
Effect size
</div>
</th>
<th style="padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="9">
<div style="border-bottom: 1px solid #00000020; padding-bottom: 5px; ">
Heterogeneity
</div>
</th>
</tr>
<tr>
<th style="text-align:left;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:center;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:center;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:center;font-style: italic;font-size: 14px;">
r
</th>
<th style="text-align:right;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:left;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:right;font-style: italic;font-size: 14px;">
P<sub>MCMC</sub>
</th>
<th style="text-align:center;font-style: italic;font-size: 14px;">
I²<sub>Phylogeny</sub>
</th>
<th style="text-align:right;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:left;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:center;font-style: italic;font-size: 14px;">
I²<sub>Study</sub>
</th>
<th style="text-align:right;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:left;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:center;font-style: italic;font-size: 14px;">
I²<sub>Observation</sub>
</th>
<th style="text-align:right;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:left;font-style: italic;font-size: 14px;">
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Global model (non-phylogenetic)
</td>
<td style="text-align:center;">
111
</td>
<td style="text-align:center;">
72
</td>
<td style="text-align:center;">
0.37
</td>
<td style="text-align:right;">
(0.37,
</td>
<td style="text-align:left;">
0.47)
</td>
<td style="text-align:right;">
&lt;0.001
</td>
<td style="text-align:center;">
–
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:center;">
–
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
<td style="text-align:center;">
–
</td>
<td style="text-align:right;">
</td>
<td style="text-align:left;">
</td>
</tr>
<tr>
<td style="text-align:left;">
Global model (phylogenetic)
</td>
<td style="text-align:center;">
111
</td>
<td style="text-align:center;">
72
</td>
<td style="text-align:center;">
0.28
</td>
<td style="text-align:right;">
(0.24,
</td>
<td style="text-align:left;">
0.41)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.57
</td>
<td style="text-align:right;">
(0.13,
</td>
<td style="text-align:left;">
0.78)
</td>
<td style="text-align:center;">
0.29
</td>
<td style="text-align:right;">
(0,
</td>
<td style="text-align:left;">
0.6)
</td>
<td style="text-align:center;">
0.06
</td>
<td style="text-align:right;">
(0,
</td>
<td style="text-align:left;">
0.35)
</td>
</tr>
<tr>
<td style="text-align:left;">
Copulatory mating success
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
21
</td>
<td style="text-align:center;">
0.20
</td>
<td style="text-align:right;">
(0.06,
</td>
<td style="text-align:left;">
0.27)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.7
</td>
<td style="text-align:right;">
(0.02,
</td>
<td style="text-align:left;">
0.77)
</td>
<td style="text-align:center;">
0.06
</td>
<td style="text-align:right;">
(0,
</td>
<td style="text-align:left;">
0.3)
</td>
<td style="text-align:center;">
0.48
</td>
<td style="text-align:right;">
(0.02,
</td>
<td style="text-align:left;">
0.61)
</td>
</tr>
<tr>
<td style="text-align:left;">
Genetic mating success
</td>
<td style="text-align:center;">
74
</td>
<td style="text-align:center;">
52
</td>
<td style="text-align:center;">
0.45
</td>
<td style="text-align:right;">
(0.2,
</td>
<td style="text-align:left;">
0.58)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.17
</td>
<td style="text-align:right;">
(0.01,
</td>
<td style="text-align:left;">
0.83)
</td>
<td style="text-align:center;">
0.75
</td>
<td style="text-align:right;">
(0,
</td>
<td style="text-align:left;">
0.83)
</td>
<td style="text-align:center;">
0.1
</td>
<td style="text-align:right;">
(0.04,
</td>
<td style="text-align:left;">
0.3)
</td>
</tr>
<tr>
<td style="text-align:left;">
Including zero mating success
</td>
<td style="text-align:center;">
61
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
0.52
</td>
<td style="text-align:right;">
(0.11,
</td>
<td style="text-align:left;">
0.58)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.54
</td>
<td style="text-align:right;">
(0.52,
</td>
<td style="text-align:left;">
0.91)
</td>
<td style="text-align:center;">
0.01
</td>
<td style="text-align:right;">
(0.01,
</td>
<td style="text-align:left;">
0.36)
</td>
<td style="text-align:center;">
0.01
</td>
<td style="text-align:right;">
(0.01,
</td>
<td style="text-align:left;">
0.14)
</td>
</tr>
<tr>
<td style="text-align:left;">
Excluding zero mating success
</td>
<td style="text-align:center;">
50
</td>
<td style="text-align:center;">
41
</td>
<td style="text-align:center;">
0.34
</td>
<td style="text-align:right;">
(0.05,
</td>
<td style="text-align:left;">
0.41)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.07
</td>
<td style="text-align:right;">
(0.05,
</td>
<td style="text-align:left;">
0.48)
</td>
<td style="text-align:center;">
0.62
</td>
<td style="text-align:right;">
(0.02,
</td>
<td style="text-align:left;">
0.84)
</td>
<td style="text-align:center;">
0.01
</td>
<td style="text-align:right;">
(0.01,
</td>
<td style="text-align:left;">
0.58)
</td>
</tr>
<tr>
<td style="text-align:left;">
Laboratory studies
</td>
<td style="text-align:center;">
46
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
0.36
</td>
<td style="text-align:right;">
(0.16,
</td>
<td style="text-align:left;">
0.5)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.68
</td>
<td style="text-align:right;">
(0.25,
</td>
<td style="text-align:left;">
0.88)
</td>
<td style="text-align:center;">
0.03
</td>
<td style="text-align:right;">
(0.01,
</td>
<td style="text-align:left;">
0.33)
</td>
<td style="text-align:center;">
0.03
</td>
<td style="text-align:right;">
(0,
</td>
<td style="text-align:left;">
0.51)
</td>
</tr>
<tr>
<td style="text-align:left;">
Field studies
</td>
<td style="text-align:center;">
65
</td>
<td style="text-align:center;">
45
</td>
<td style="text-align:center;">
0.47
</td>
<td style="text-align:right;">
(0.24,
</td>
<td style="text-align:left;">
0.61)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.01
</td>
<td style="text-align:right;">
(0.01,
</td>
<td style="text-align:left;">
0.84)
</td>
<td style="text-align:center;">
0.82
</td>
<td style="text-align:right;">
(0.11,
</td>
<td style="text-align:left;">
0.9)
</td>
<td style="text-align:center;">
0.02
</td>
<td style="text-align:right;">
(0,
</td>
<td style="text-align:left;">
0.08)
</td>
</tr>
<tr>
<td style="text-align:left;">
monandrous species
</td>
<td style="text-align:center;">
32
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
0.22
</td>
<td style="text-align:right;">
(0.09,
</td>
<td style="text-align:left;">
0.31)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.1
</td>
<td style="text-align:right;">
(0.01,
</td>
<td style="text-align:left;">
0.57)
</td>
<td style="text-align:center;">
0.29
</td>
<td style="text-align:right;">
(0.07,
</td>
<td style="text-align:left;">
0.87)
</td>
<td style="text-align:center;">
0.06
</td>
<td style="text-align:right;">
(0.01,
</td>
<td style="text-align:left;">
0.59)
</td>
</tr>
<tr>
<td style="text-align:left;">
Polyandrous species
</td>
<td style="text-align:center;">
79
</td>
<td style="text-align:center;">
56
</td>
<td style="text-align:center;">
0.42
</td>
<td style="text-align:right;">
(0.2,
</td>
<td style="text-align:left;">
0.66)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.69
</td>
<td style="text-align:right;">
(0.21,
</td>
<td style="text-align:left;">
0.94)
</td>
<td style="text-align:center;">
0.04
</td>
<td style="text-align:right;">
(0,
</td>
<td style="text-align:left;">
0.58)
</td>
<td style="text-align:center;">
0.03
</td>
<td style="text-align:right;">
(0,
</td>
<td style="text-align:left;">
0.21)
</td>
</tr>
</tbody>
</table>
<p><br></br> <br></br> <strong>Table 2. Predictors of inter-specific variation in female Bateman gradients.</strong> Methodological moderators include mating success method (copulatory versus genetic mating success), mating success range (including versus excluding zero mating success category), study type (field versus lab) and year of publication (continuous variable). Effect of mating system contrasts polyandrous and monandrous species. Effect of polyandry estimates the relationship between the female Bateman gradient and the proportion of polyandrous females in the population. Model estimates (i.e., estimated difference between groups) are shown as posterior modes with 95% Highest Posterior Density (HPD) intervals obtained from phylogenetically controlled General Linear-Mixed Effects Models. The variance explained by the moderator variable is given as the marginal R2 with 95% HPD intervals in brackets.</p>
<pre class="r"><code>for (i in c(2, 6)){
  Table2[, i] &lt;- paste0(&quot;(&quot;, Table2[, i], &quot;, &quot;)}
for (i in c(3, 7)){
  Table2[, i] &lt;- paste0(Table2[, i], &quot;)&quot;)}

# All predictor models summary
rownames(Table2) &lt;- c(&quot;Mating success method&quot;,
                      &quot;Mating success range&quot;,
                      &quot;Study type&quot;,
                      &quot;Year of publication&quot;,
                      &quot;Mating system&quot;,
                      &quot;Polyandry&quot;)
#Table2[1, 6] &lt;- &quot;&lt;0.001&quot;

Table2 %&gt;%
kable(align = c(&quot;c&quot;, &quot;r&quot;, &quot;l&quot;, &quot;r&quot;, &quot;c&quot;, &quot;r&quot;, &quot;l&quot;),
      col.names = c(&quot;Estimate&quot;, &quot;&quot;, &quot;&quot;, &quot;P~MCMC~&quot;, 
                      &quot;R²&quot;, &quot;&quot;, &quot;&quot;)) %&gt;% 
  kable_paper(full_width = FALSE)</code></pre>
<table class=" lightable-paper" style='font-family: "Arial Narrow", arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto;'>
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:center;">
Estimate
</th>
<th style="text-align:right;">
</th>
<th style="text-align:left;">
</th>
<th style="text-align:right;">
P<sub>MCMC</sub>
</th>
<th style="text-align:center;">
R²
</th>
<th style="text-align:right;">
</th>
<th style="text-align:left;">
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Mating success method
</td>
<td style="text-align:center;">
0.21
</td>
<td style="text-align:right;">
(0.11,
</td>
<td style="text-align:left;">
0.45)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.08
</td>
<td style="text-align:right;">
(0.07,
</td>
<td style="text-align:left;">
0.16)
</td>
</tr>
<tr>
<td style="text-align:left;">
Mating success range
</td>
<td style="text-align:center;">
0.20
</td>
<td style="text-align:right;">
(0.08,
</td>
<td style="text-align:left;">
0.31)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.06
</td>
<td style="text-align:right;">
(0.04,
</td>
<td style="text-align:left;">
0.11)
</td>
</tr>
<tr>
<td style="text-align:left;">
Study type
</td>
<td style="text-align:center;">
0.05
</td>
<td style="text-align:right;">
(-0.09,
</td>
<td style="text-align:left;">
0.08)
</td>
<td style="text-align:right;">
1.0
</td>
<td style="text-align:center;">
0.00
</td>
<td style="text-align:right;">
(0,
</td>
<td style="text-align:left;">
<ol start="0" style="list-style-type: decimal">
<li></td>
</tr>
<tr>
<td style="text-align:left;">
Year of publication
</td>
<td style="text-align:center;">
0.00
</td>
<td style="text-align:right;">
(-0.02,
</td>
<td style="text-align:left;">
0.01)
</td>
<td style="text-align:right;">
0.6
</td>
<td style="text-align:center;">
0.00
</td>
<td style="text-align:right;">
(0,
</td>
<td style="text-align:left;">
<ol start="0" style="list-style-type: decimal">
<li></td>
</tr>
<tr>
<td style="text-align:left;">
Mating system
</td>
<td style="text-align:center;">
0.25
</td>
<td style="text-align:right;">
(0.19,
</td>
<td style="text-align:left;">
0.38)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.06
</td>
<td style="text-align:right;">
(0.06,
</td>
<td style="text-align:left;">
0.16)
</td>
</tr>
<tr>
<td style="text-align:left;">
Polyandry
</td>
<td style="text-align:center;">
0.78
</td>
<td style="text-align:right;">
(0.41,
</td>
<td style="text-align:left;">
1.04)
</td>
<td style="text-align:right;">
0.1
</td>
<td style="text-align:center;">
0.22
</td>
<td style="text-align:right;">
(0.17,
</td>
<td style="text-align:left;">
0.26)
</td>
</tr>
</tbody>
</table>
<br></br> ______________<br />
# Restricted Maximum Likelihood Meta-Analysis For completeness, we also ran all GLMMs using the Restricted Maximum Likelihood (REML) approach using the <code>metafor</code> package <span class="citation">(Viechtbauer 2010)</span>.</li>
</ol></li>
</ol>
<pre class="r"><code>Data &lt;- read.table(&quot;./data/Fromonteil_et_al_DATA.txt&quot;, fileEncoding = &quot;LATIN1&quot;) # the file encoding supports special characters
Tree &lt;- read.tree(&quot;./data/Fromonteil_et_al_TREE.txt&quot;)
MetaData_All_Species_Data &lt;- unique(Data$Species_Phylo)
Tree &lt;- drop.tip(Tree, Tree$tip.label[-na.omit(match(MetaData_All_Species_Data, Tree$tip.label))])

# Run Models #### 
    # All
forcedC_ALL &lt;- as.matrix(forceSymmetric(vcv(Tree, corr = TRUE))) # formatting for the phylogenetic tree

Model_REML_ALL &lt;- rma.mv(r_f ~ 1, 
                         V = r_VAR_f, 
                         data = Data, 
                         random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo), 
                         R = list(Species_Phylo = forcedC_ALL), method = &quot;REML&quot;)

    # Mating success method
forcedC_cMS &lt;- as.matrix(forceSymmetric(vcv(Tree_cMS, corr = TRUE)))
forcedC_gMS &lt;- as.matrix(forceSymmetric(vcv(Tree_gMS, corr = TRUE)))

MetaData_cMS              &lt;- subset(Data, MS_estimate == &quot;cMS&quot;)
MetaData_gMS              &lt;- subset(Data, MS_estimate == &quot;gMS&quot;)

Model_REML_MS_Method &lt;- rma.mv(r_f ~ factor(MS_estimate), 
                               V = r_VAR_f, 
                               data = Data, 
                               random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo), 
                               R = list(Species_Phylo = forcedC_ALL), method = &quot;REML&quot;)

Model_REML_cMS &lt;- rma.mv(r_f ~ 1, 
                         V = r_VAR_f, 
                         data = MetaData_cMS, 
                         random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo),
                         R = list(Species_Phylo = forcedC_cMS), method = &quot;REML&quot;)
Model_REML_gMS &lt;- rma.mv(r_f ~ 1, 
                         V = r_VAR_f, 
                         data = MetaData_gMS, 
                         random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo),
                         R = list(Species_Phylo = forcedC_gMS), method = &quot;REML&quot;)

    # Mating success range
forcedC_with_0MS &lt;- as.matrix(forceSymmetric(vcv(Tree_with_0MS, corr = TRUE)))
forcedC_without_0MS &lt;- as.matrix(forceSymmetric(vcv(Tree_without_0MS, corr = TRUE)))

MetaData_with_0MS                 &lt;- subset(Data, Zero_f == &quot;1&quot;)
MetaData_without_0MS              &lt;- subset(Data, Zero_f == &quot;0&quot;)

Model_REML_MS_Zero &lt;- rma.mv(r_f ~ factor(Zero_f), 
                             V = r_VAR_f, 
                             data = Data, 
                             random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo), 
                             R = list(Species_Phylo = forcedC_ALL), method = &quot;REML&quot;)


Model_REML_with_0MS &lt;- rma.mv(r_f ~ 1, 
                         V = r_VAR_f, 
                         data = MetaData_with_0MS, 
                         random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo),
                         R = list(Species_Phylo = forcedC_with_0MS), method = &quot;REML&quot;)
Model_REML_without_0MS = rma.mv(r_f ~ 1, 
                         V = r_VAR_f, 
                         data = MetaData_without_0MS, 
                         random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo),
                         R = list(Species_Phylo = forcedC_without_0MS), method = &quot;REML&quot;)

    # Study type
forcedC_Field &lt;- as.matrix(forceSymmetric(vcv(Tree_Field, corr = TRUE)))
forcedC_Lab &lt;- as.matrix(forceSymmetric(vcv(Tree_Lab, corr = TRUE)))

MetaData_Field              &lt;- subset(Data, StudyType == &quot;Field&quot;)
MetaData_Lab                &lt;- subset(Data, StudyType == &quot;Lab&quot;)

Model_REML_StudyType &lt;- rma.mv(r_f ~ factor(StudyType), 
                               V = r_VAR_f, 
                               data = Data, 
                               random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo), 
                               R = list(Species_Phylo = forcedC_ALL), method = &quot;REML&quot;)

Model_REML_Field &lt;- rma.mv(r_f ~ 1, 
                         V = r_VAR_f, 
                         data = MetaData_Field, 
                         random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo),
                         R = list(Species_Phylo = forcedC_Field), method = &quot;REML&quot;)
Model_REML_Lab &lt;- rma.mv(r_f ~ 1, 
                         V = r_VAR_f, 
                         data = MetaData_Lab, 
                         random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo),
                         R = list(Species_Phylo = forcedC_Lab), method = &quot;REML&quot;)

    # Mating system
forcedC_polygamous &lt;- as.matrix(forceSymmetric(vcv(Tree_polygamous, corr = TRUE)))
forcedC_monogamous &lt;- as.matrix(forceSymmetric(vcv(Tree_monogamous, corr = TRUE)))

MetaData_polygamous              &lt;- subset(Data, MatingSystem == &quot;polyandrous&quot;)
MetaData_monogamous              &lt;- subset(Data, MatingSystem == &quot;monogamous&quot;)

Model_REML_MatingSystem &lt;- rma.mv(r_f ~ factor(MatingSystem), 
                                           V = r_VAR_f, 
                                           data = Data, 
                                           random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo), 
                                           R = list(Species_Phylo = forcedC_ALL), method = &quot;REML&quot;)

Model_REML_polygamous &lt;- rma.mv(r_f ~ 1, 
                         V = r_VAR_f, 
                         data = MetaData_polygamous, 
                         random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo),
                         R = list(Species_Phylo = forcedC_polygamous), method = &quot;REML&quot;)
Model_REML_monogamous &lt;- rma.mv(r_f ~ 1, 
                         V = r_VAR_f, 
                         data = MetaData_monogamous, 
                         random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo),
                         R = list(Species_Phylo = forcedC_monogamous), method = &quot;REML&quot;)

    # Polyandry
Model_REML_Polyandry &lt;- rma.mv(r_f ~ Polyandry, 
                               V = r_VAR_f, 
                               data = Data, 
                               random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo), 
                               R = list(Species_Phylo = forcedC_ALL), method = &quot;REML&quot;)

    # Year
Model_REML_Year &lt;- rma.mv(r_f ~ Year, 
                          V = r_VAR_f, 
                          data = Data, 
                          random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo), 
                          R = list(Species_Phylo = forcedC_ALL), method = &quot;REML&quot;)</code></pre>
</div>
<div id="models-summary-1" class="section level2">
<h2>Models summary</h2>
<p><strong>Table S1.</strong> Global tests of sexual selection in females using the restricted maximum likelihood (REML) approach. Results of intercept-only phylogenetically controlled General Linear-Mixed Effects Models are shown for the entire dataset (global model) and subsets with respect to mating success method (copulatory versus genetic), mating success range (including versus excluding zero mating success category), study type (laboratory versus field studies) and mating system (monandrous versus polygamous species). Table shows number of effect sizes (<em>k</em>), number of species (<em>N</em>) and estimates of r together with 95% confidence intervals (in brackets).</p>
<pre class="r"><code>REMLNullModel_summary1 &lt;- function(which_data) {
  k &lt;- length(which_data$r_f)  # sample size
  N_sp &lt;- length((aggregate(r_f ~ Species_Phylo, data = which_data, FUN = sum))$Species_Phylo)  # number of species
  
  as.data.frame(cbind(k, N_sp))
}

REMLNullModel_summary2 &lt;- function(which_model) {
  # REML summary
  as.data.frame(cbind(
    summary(which_model)$&quot;beta&quot;,  # global effect size
    summary(which_model)$ci.lb, # 95% confidence interval - low
    summary(which_model)$ci.ub, # 95% confidence interval - high
    summary(which_model)$zval, # Z-value
    summary(which_model)$pval)) # P-value

}

List_data &lt;- list(Data, 
                  MetaData_cMS, MetaData_gMS, 
                  MetaData_with_0MS, MetaData_without_0MS, 
                  MetaData_Lab, MetaData_Field, 
                  MetaData_monogamous, MetaData_polygamous)

List_Nullmodels &lt;- list(Model_REML_ALL,
                        Model_REML_cMS, Model_REML_gMS,
                        Model_REML_with_0MS, Model_REML_without_0MS,
                        Model_REML_Lab, Model_REML_Field,
                        Model_REML_monogamous, Model_REML_polygamous)
Model_sum1 &lt;- adply(.data = List_data, .margins = 1, .fun = REMLNullModel_summary1)
Model_sum2 &lt;- adply(.data = List_Nullmodels, .margins = 1, .fun = REMLNullModel_summary2)


TableS1 &lt;- cbind(Model_sum1[,-1], Model_sum2[, -1]) %&gt;%
mutate_at(c(3:5), round, digits = 2)
TableS1 &lt;- TableS1 %&gt;%
  mutate_at(6:7, round, digits = 3)
for (i in c(4)){
  TableS1[, i] &lt;- paste0(&quot;(&quot;, TableS1[, i], &quot;, &quot;)}
for (i in c(5)){
  TableS1[, i] &lt;- paste0(TableS1[, i], &quot;)&quot;)}

colnames(TableS1) &lt;- c(&quot;&quot;, &quot;&quot;, &quot;r&quot;, &quot;&quot;, &quot;&quot;, &quot;z-value&quot;, &quot;P-value&quot;)
rownames(TableS1) &lt;- c(&quot;Global model&quot;,
                       &quot;Copulatory mating success&quot;,
                       &quot;Genetic mating success&quot;,
                       &quot;Including zero mating success&quot;,
                       &quot;Excluding zero mating success&quot;,
                       &quot;Laboratory studies&quot;,
                       &quot;Field studies&quot;,
                       &quot;monandrous species&quot;,
                       &quot;Polyandrous species&quot;)
#TableS1[1, 6] &lt;- &quot;&lt;0.001&quot;

TableS1 %&gt;%
kable(align = c(rep(&quot;c&quot;, 3), &quot;r&quot;, &quot;l&quot;, &quot;c&quot;, &quot;c&quot;)) %&gt;%
  kable_paper(full_width = T) %&gt;%
  add_header_above(c(&quot;Model&quot; = 1, &quot;k&quot; = 1, &quot;N~Species~&quot; = 1, &quot;Global effect size&quot; = 3, &quot; &quot; = 2)) %&gt;%
  row_spec(0, italic = T, font_size = 14)</code></pre>
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="1">
<div style="border-bottom: 1px solid #00000020; padding-bottom: 5px; ">
Model
</div>
</th>
<th style="padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="1">
<div style="border-bottom: 1px solid #00000020; padding-bottom: 5px; ">
k
</div>
</th>
<th style="padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="1">
<div style="border-bottom: 1px solid #00000020; padding-bottom: 5px; ">
N<sub>Species</sub>
</div>
</th>
<th style="padding-bottom:0; padding-left:3px;padding-right:3px;text-align: center; " colspan="3">
<div style="border-bottom: 1px solid #00000020; padding-bottom: 5px; ">
Global effect size
</div>
</th>
<th style="empty-cells: hide;" colspan="2">
</th>
</tr>
<tr>
<th style="text-align:left;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:center;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:center;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:center;font-style: italic;font-size: 14px;">
r
</th>
<th style="text-align:right;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:left;font-style: italic;font-size: 14px;">
</th>
<th style="text-align:center;font-style: italic;font-size: 14px;">
z-value
</th>
<th style="text-align:center;font-style: italic;font-size: 14px;">
P-value
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Global model
</td>
<td style="text-align:center;">
111
</td>
<td style="text-align:center;">
72
</td>
<td style="text-align:center;">
0.37
</td>
<td style="text-align:right;">
(0.11,
</td>
<td style="text-align:left;">
0.62)
</td>
<td style="text-align:center;">
2.820
</td>
<td style="text-align:center;">
0.005
</td>
</tr>
<tr>
<td style="text-align:left;">
Copulatory mating success
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
21
</td>
<td style="text-align:center;">
0.21
</td>
<td style="text-align:right;">
(0.06,
</td>
<td style="text-align:left;">
0.35)
</td>
<td style="text-align:center;">
2.854
</td>
<td style="text-align:center;">
0.004
</td>
</tr>
<tr>
<td style="text-align:left;">
Genetic mating success
</td>
<td style="text-align:center;">
74
</td>
<td style="text-align:center;">
52
</td>
<td style="text-align:center;">
0.47
</td>
<td style="text-align:right;">
(0.25,
</td>
<td style="text-align:left;">
0.68)
</td>
<td style="text-align:center;">
4.237
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:left;">
Including zero mating success
</td>
<td style="text-align:center;">
61
</td>
<td style="text-align:center;">
37
</td>
<td style="text-align:center;">
0.44
</td>
<td style="text-align:right;">
(0.15,
</td>
<td style="text-align:left;">
0.74)
</td>
<td style="text-align:center;">
2.996
</td>
<td style="text-align:center;">
0.003
</td>
</tr>
<tr>
<td style="text-align:left;">
Excluding zero mating success
</td>
<td style="text-align:center;">
50
</td>
<td style="text-align:center;">
41
</td>
<td style="text-align:center;">
0.28
</td>
<td style="text-align:right;">
(0.14,
</td>
<td style="text-align:left;">
0.42)
</td>
<td style="text-align:center;">
3.906
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:left;">
Laboratory studies
</td>
<td style="text-align:center;">
46
</td>
<td style="text-align:center;">
28
</td>
<td style="text-align:center;">
0.39
</td>
<td style="text-align:right;">
(0.11,
</td>
<td style="text-align:left;">
0.66)
</td>
<td style="text-align:center;">
2.766
</td>
<td style="text-align:center;">
0.006
</td>
</tr>
<tr>
<td style="text-align:left;">
Field studies
</td>
<td style="text-align:center;">
65
</td>
<td style="text-align:center;">
45
</td>
<td style="text-align:center;">
0.40
</td>
<td style="text-align:right;">
(0.3,
</td>
<td style="text-align:left;">
0.49)
</td>
<td style="text-align:center;">
8.415
</td>
<td style="text-align:center;">
0.000
</td>
</tr>
<tr>
<td style="text-align:left;">
monandrous species
</td>
<td style="text-align:center;">
32
</td>
<td style="text-align:center;">
16
</td>
<td style="text-align:center;">
0.23
</td>
<td style="text-align:right;">
(0.09,
</td>
<td style="text-align:left;">
0.37)
</td>
<td style="text-align:center;">
3.162
</td>
<td style="text-align:center;">
0.002
</td>
</tr>
<tr>
<td style="text-align:left;">
Polyandrous species
</td>
<td style="text-align:center;">
79
</td>
<td style="text-align:center;">
56
</td>
<td style="text-align:center;">
0.40
</td>
<td style="text-align:right;">
(0.12,
</td>
<td style="text-align:left;">
0.68)
</td>
<td style="text-align:center;">
2.817
</td>
<td style="text-align:center;">
0.005
</td>
</tr>
</tbody>
</table>
<p><br></br> <br></br> <strong>Table S2.</strong> Predictors of inter-specific variation in female Bateman gradients using restricted maximum likelihood (REML) approach. Methodological moderators include mating success method (copulatory versus genetic mating success), mating success range (including versus excluding mating success category), study type (field versus lab) and year of publication (continuous variable). Effect of mating system contrasts polyandrous and monandrous species. Effect of polyandry (continuous variable) estimates the relationship between the female Bateman gradient and the proportion of polyandrous females in the population. Phylogenetically controlled multilevel meta-analytic single predictor models are shown with Omnibus tests (Wald-type chi-square test) and McFadden’s <em>R²</em>.</p>
<pre class="r"><code>REMLPredictorModel_summary &lt;- function(which_model) {
  # REML summary
  as.data.frame(cbind(
    coef(summary(which_model))[2, 1],  # global effect size
    coef(summary(which_model))[2, 2], # standard error
    summary(which_model)$&quot;QM&quot;, # QM: Omnibus test
    summary(which_model)$&quot;QMp&quot;, # P-value of QM
    1 - (logLik.rma(which_model)/logLik.rma(Model_REML_ALL)))) # R²: McFadden&#39;s R2
}


List_Predmodels &lt;- list(Model_REML_MS_Method,
                        Model_REML_MS_Zero,
                        Model_REML_StudyType,
                        Model_REML_Year, 
                        Model_REML_MatingSystem,
                        Model_REML_Polyandry)
Model_sum &lt;- adply(List_Predmodels, 1, REMLPredictorModel_summary)


TableS2 &lt;- cbind(Model_sum[,-1]) %&gt;%
mutate_at(c(1:3), round, digits = 2)
TableS2 &lt;- TableS2 %&gt;%
  mutate_at(4, round, digits = 3)
for (i in c(2)){
  TableS2[, i] &lt;- paste0(&quot;±&quot;, TableS2[, i])}

colnames(TableS2) &lt;- c(&quot;Estimate&quot;, &quot;± SE&quot;, &quot;Q~M~&quot;, &quot;P-value&quot;, &quot;R²&quot;)
rownames(TableS2) &lt;- c(&quot;Mating success method&quot;,
                       &quot;Mating success range&quot;,
                       &quot;Study type&quot;,
                       &quot;Year&quot;,
                       &quot;Mating system&quot;,
                       &quot;Polyandry&quot;)
#TableS1[1, 6] &lt;- &quot;&lt;0.001&quot;

TableS2 %&gt;%
kable(align = c(&quot;r&quot;, &quot;l&quot;, &quot;c&quot;, &quot;c&quot;)) %&gt;%
  kable_paper(full_width = F)</code></pre>
<table class=" lightable-paper" style="font-family: &quot;Arial Narrow&quot;, arial, helvetica, sans-serif; width: auto !important; margin-left: auto; margin-right: auto;">
<thead>
<tr>
<th style="text-align:left;">
</th>
<th style="text-align:right;">
Estimate
</th>
<th style="text-align:left;">
± SE
</th>
<th style="text-align:center;">
Q<sub>M</sub>
</th>
<th style="text-align:center;">
P-value
</th>
<th style="text-align:right;">
R²
</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:left;">
Mating success method
</td>
<td style="text-align:right;">
0.30
</td>
<td style="text-align:left;">
±0.09
</td>
<td style="text-align:center;">
11.93
</td>
<td style="text-align:center;">
0.001
</td>
<td style="text-align:right;">
0.2279309
</td>
</tr>
<tr>
<td style="text-align:left;">
Mating success range
</td>
<td style="text-align:right;">
0.18
</td>
<td style="text-align:left;">
±0.06
</td>
<td style="text-align:center;">
7.95
</td>
<td style="text-align:center;">
0.005
</td>
<td style="text-align:right;">
0.1651374
</td>
</tr>
<tr>
<td style="text-align:left;">
Study type
</td>
<td style="text-align:right;">
0.06
</td>
<td style="text-align:left;">
±0.1
</td>
<td style="text-align:center;">
0.39
</td>
<td style="text-align:center;">
0.533
</td>
<td style="text-align:right;">
0.0193860
</td>
</tr>
<tr>
<td style="text-align:left;">
Year
</td>
<td style="text-align:right;">
-0.01
</td>
<td style="text-align:left;">
±0.01
</td>
<td style="text-align:center;">
0.83
</td>
<td style="text-align:center;">
0.362
</td>
<td style="text-align:right;">
0.0143516
</td>
</tr>
<tr>
<td style="text-align:left;">
Mating system
</td>
<td style="text-align:right;">
0.31
</td>
<td style="text-align:left;">
±0.08
</td>
<td style="text-align:center;">
16.35
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:right;">
0.3205866
</td>
</tr>
<tr>
<td style="text-align:left;">
Polyandry
</td>
<td style="text-align:right;">
0.67
</td>
<td style="text-align:left;">
±0.14
</td>
<td style="text-align:center;">
22.61
</td>
<td style="text-align:center;">
0.000
</td>
<td style="text-align:right;">
0.4350441
</td>
</tr>
</tbody>
</table>
<p><br></br></p>
<p>______________<br />
# Publication bias Lastly, we build funnel plots to visually assess asymmetry.</p>
<pre class="r"><code># Funnel plot for raw Bateman gradients
  # Define model
forcedC &lt;- as.matrix(forceSymmetric(vcv(Tree, corr = TRUE)))

Model_REML_ALL_by_Moderator &lt;- rma.mv(r_f ~ factor(MatingSystem) + factor(MS_estimate) + factor(Zero_f) + factor(StudyType) + Year, 
                                      V = r_VAR_f, 
                                      data = Data, 
                                      random = c(~ 1 | Study_ID, ~ 1 | Index, ~ 1 | Species_Phylo), 
                                      R = list(Species_Phylo = forcedC), method = &quot;REML&quot;)

Data$Residuals_Phylo &lt;- resid(Model_REML_ALL)
Data$Residuals_Phylo_Moderator &lt;- resid(Model_REML_ALL_by_Moderator)


  # extract relevant information
Data$r_SD &lt;- sqrt(Data$r_VAR_f) # standard deviation
Data$r_SE &lt;- Data$r_SD / sqrt(Data$N_f) # standard error
effect_size_mode &lt;- posterior.mode(Model_ALL_phylo$Sol[, &quot;(Intercept)&quot;]) # mode
effect_size_ll95 &lt;- summary(Model_ALL_phylo)$solutions[, 2] # lower 95% highest posterior density
effect_size_ul95 &lt;- summary(Model_ALL_phylo)$solutions[, 3] # upper 95% highest posterior density

  # Additional lines on funnel plot
se.seq &lt;- seq(0, max(Data$r_SE), 0.001) # range from 0 to the max SE in the data

    # 95% confidence interval
ll95 &lt;- effect_size_mode - (1.96*se.seq)
ul95 &lt;- effect_size_mode + (1.96*se.seq)
    # 99% confidence interval
ll99 &lt;- effect_size_mode - (3.29*se.seq)
ul99 &lt;- effect_size_mode + (3.29*se.seq)


dfCI &lt;- data.frame(ll95, ul95, 
                   ll99, ul99, 
                   se.seq, effect_size_mode, 
                   effect_size_ll95, effect_size_ul95)

# plot
FunnelPlot &lt;- ggplot() +
  geom_line(data = dfCI,
            mapping = aes(x = se.seq, y = ll95), 
            linetype = &quot;solid&quot;, colour = &quot;#B2D8D8&quot;, size = 1) +
  geom_line(data = dfCI,
            mapping = aes(x = se.seq, y = ul95), 
            linetype = &quot;solid&quot;, colour = &quot;#B2D8D8&quot;, size = 1) +
  geom_line(data = dfCI,
            mapping = aes(x = se.seq, y = ll99), 
            linetype = &quot;solid&quot;, colour = &quot;#008080&quot;, size = 1) +
  geom_line(data = dfCI,
            mapping = aes(x = se.seq, y = ul99), 
            linetype = &quot;solid&quot;, colour = &quot;#008080&quot;, size = 1) +
  geom_segment(data = dfCI,
               mapping = aes(x = min(se.seq), y = effect_size_mode, xend = max(se.seq), yend = effect_size_mode),
               linetype = &quot;dashed&quot;, size = 0.7) +
  geom_point(data = Data,
             mapping = aes(x = r_SE, y = r_f),
             shape = 21, size = 4, fill = &quot;grey&quot;, alpha = .6) +
  labs(title = &quot;&quot;, x = &quot;Standard error&quot;, y = &quot;Bateman gradient (*r*)&quot;) +
  scale_y_continuous(breaks = seq(min(Data$Residuals_Phylo_Moderator) - 0.1, max(Data$r_f) + 0.1, 0.5), 
                     labels = scales::number_format(accuracy = 0.1),
                     limits = c(min(Data$Residuals_Phylo_Moderator) - 0.1, max(Data$r_f) + 0.1)) +
  scale_x_reverse() +
  coord_flip() +
  theme_paper(axis.title.x = element_markdown())


# Funnel plot for residual Bateman Gradients
### Customized Egger&#39;s regression ###
Data$Precision &lt;- 1/(Data$r_SE)
Data$SND_NullModel &lt;- Data$Residuals_Phylo/(Data$r_SE)
Data$SND_ModeratorModel &lt;- Data$Residuals_Phylo_Moderator/(Data$r_SE)

estimate_RES &lt;- mean(Data$Residuals_Phylo_Moderator)
se &lt;- sd(Data$Residuals_Phylo_Moderator)/sqrt(length(Data$Residuals_Phylo_Moderator))

# Additional lines on res funnel plot
  # 95% confidence interval
Res_ll95 = 0 - (1.96*se.seq)
Res_ul95 = 0 + (1.96*se.seq)

  # 99% confidence interval
Res_ll99 = 0 - (3.29*se.seq)
Res_ul99 = 0 + (3.29*se.seq)

# confidence interval for the meta-analytic estimate based on the stored value of its standard error
Res_meanll95 = 0 - (1.96*se)
Res_meanul95 = 0 + (1.96*se)

Res_dfCI &lt;- data.frame(Res_ll95, Res_ul95, 
                      Res_ll99, Res_ul99, 
                      se.seq, estimate_RES, 
                      Res_meanll95, Res_meanul95)


# plot
FunnelPlot_Resid &lt;- ggplot() +
  geom_line(data = Res_dfCI,
            mapping = aes(x = se.seq, y = Res_ll95), 
            linetype = &quot;solid&quot;, colour = &quot;#B2D8D8&quot;, size = 1) +
  geom_line(data = Res_dfCI,
            mapping = aes(x = se.seq, y = Res_ul95), 
            linetype = &quot;solid&quot;, colour = &quot;#B2D8D8&quot;, size = 1) +
  geom_line(data = Res_dfCI,
            mapping = aes(x = se.seq, y = Res_ll99), 
            linetype = &quot;solid&quot;, colour = &quot;#008080&quot;, size = 1) +
  geom_line(data = Res_dfCI,
            mapping = aes(x = se.seq, y = Res_ul99), 
            linetype = &quot;solid&quot;, colour = &quot;#008080&quot;, size = 1) +
  geom_segment(data = Res_dfCI,
            mapping = aes(x = min(se.seq), y = 0, xend = max(se.seq), yend = 0),
            linetype = &quot;dashed&quot;, size = 0.7) +
  geom_point(data = Data,
             mapping = aes(x = r_SE, y = Residuals_Phylo_Moderator),
             shape = 21, size = 4, fill = &quot;grey&quot;, alpha = .6) +
  labs(title = &quot;&quot;, x = &quot;Standard error&quot;, y = &quot;Residual Bateman gradient (*r*)&quot;) +
  scale_y_continuous(breaks = seq(min(Data$Residuals_Phylo_Moderator) - 0.1, max(Data$r_f) + 0.1, 0.5), 
                     labels = scales::number_format(accuracy = 0.1),
                     limits = c(min(Data$Residuals_Phylo_Moderator) - 0.1, max(Data$r_f) + 0.1)) +
  scale_x_reverse() +
  coord_flip() +
  theme_paper(axis.title.x = element_markdown())


Figure_FunnelPlots &lt;- plot_grid(FunnelPlot, FunnelPlot_Resid,
                                labels = &quot;AUTO&quot;,
                                label_size = 15,
                                hjust = 0, 
                                vjust = 1, 
                                align = &quot;hv&quot;,
                                ncol = 1,
                                nrow = 2)
Figure_FunnelPlots</code></pre>
<p><img src="figure/META_SexSelFem.Rmd/Funnel_plot-1.png" width="576" style="display: block; margin: auto;" /> <strong>Figure S4.</strong> Funnel plots. Data are shown for (A) raw values and (B) for meta-analytic residuals obtained from multivariate linear mixed-effects models accounting for all tested moderator variables. Dashed lines indicate the estimated global effect size. Dark and light blue solid lines denote the expected 95% and 99% confidence limits purely due to sampling heterogeneity. Asymmetries along the global effect size may reflect publication biases.</p>
<p><br></br> ______________<br />
# References</p>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-hadfield2010mcmcglmm" class="csl-entry">
Hadfield, Jarrod D. 2010. <span>“MCMC Methods for Multi-Response Generalized Linear Mixed Models: The MCMCglmm r Package.”</span> <em>Journal of Statistical Software</em> 33: 1–22.
</div>
<div id="ref-higgins2002heterogeneity" class="csl-entry">
Higgins, Julian PT, and Simon G Thompson. 2002. <span>“Quantifying Heterogeneity in a Meta-Analysis.”</span> <em>Statistics in Medicine</em> 21 (11): 1539–58.
</div>
<div id="ref-kumar2017timetree" class="csl-entry">
Kumar, Sudhir, Glen Stecher, Michael Suleski, and S Blair Hedges. 2017. <span>“TimeTree: A Resource for Timelines, Timetrees, and Divergence Times.”</span> <em>Molecular Biology and Evolution</em> 34 (7): 1812–19.
</div>
<div id="ref-nakagawa2012metaanalysis" class="csl-entry">
Nakagawa, Shinichi, and Eduardo SA Santos. 2012. <span>“Methodological Issues and Advances in Biological Meta-Analysis.”</span> <em>Evolutionary Ecology</em> 26 (5): 1253–74.
</div>
<div id="ref-nakagawa2013glmm" class="csl-entry">
Nakagawa, Shinichi, and Holger Schielzeth. 2013. <span>“A General and Simple Method for Obtaining R2 from Generalized Linear Mixed-Effects Models.”</span> <em>Methods in Ecology and Evolution</em> 4 (2): 133–42.
</div>
<div id="ref-viechtbauer2010metafor" class="csl-entry">
Viechtbauer, Wolfgang. 2010. <span>“Conducting Meta-Analyses in r with the Metafor Package.”</span> <em>Journal of Statistical Software</em> 36 (3): 1–48.
</div>
<div id="ref-yu2020ggtree" class="csl-entry">
Yu, Guangchuang. 2020. <span>“Using Ggtree to Visualize Data on Tree-Like Structures.”</span> <em>Current Protocols in Bioinformatics</em> 69 (1): e96.
</div>
<div id="ref-yu2018ggtree" class="csl-entry">
Yu, Guangchuang, Tommy Tsan-Yuk Lam, Huachen Zhu, and Yi Guan. 2018. <span>“Two Methods for Mapping and Visualizing Associated Data on Phylogeny Using Ggtree.”</span> <em>Molecular Biology and Evolution</em> 35 (12): 3041–43.
</div>
<div id="ref-yu2017ggtree" class="csl-entry">
Yu, Guangchuang, David K Smith, Huachen Zhu, Yi Guan, and Tommy Tsan-Yuk Lam. 2017. <span>“Ggtree: An r Package for Visualization and Annotation of Phylogenetic Trees with Their Covariates and Other Associated Data.”</span> <em>Methods in Ecology and Evolution</em> 8 (1): 28–36.
</div>
</div>
<br>
<p>
<button type="button" class="btn btn-default btn-workflowr btn-workflowr-sessioninfo" data-toggle="collapse" data-target="#workflowr-sessioninfo" style="display: block;">
<span class="glyphicon glyphicon-wrench" aria-hidden="true"></span> Session information
</button>
</p>
<div id="workflowr-sessioninfo" class="collapse">
<pre class="r"><code>sessionInfo()</code></pre>
<pre><code>R version 4.1.1 (2021-08-10)
Platform: i386-w64-mingw32/i386 (32-bit)
Running under: Windows 10 x64 (build 19042)

Matrix products: default

locale:
[1] LC_COLLATE=English_United Kingdom.1252 
[2] LC_CTYPE=English_United Kingdom.1252   
[3] LC_MONETARY=English_United Kingdom.1252
[4] LC_NUMERIC=C                           
[5] LC_TIME=English_United Kingdom.1252    

attached base packages:
[1] grid      stats     graphics  grDevices utils     datasets  methods  
[8] base     

other attached packages:
 [1] knitr_1.36         htmlTable_2.2.1    kableExtra_1.3.4   ggtext_0.1.1      
 [5] ggnewscale_0.4.5   RColorBrewer_1.1-2 ggtree_3.0.4       data.table_1.14.2 
 [9] dplyr_1.0.7        plyr_1.8.6         cowplot_1.1.1      ggplot2_3.3.5     
[13] MCMCglmm_2.32      coda_0.19-4        ape_5.5            metafor_3.0-2     
[17] Matrix_1.3-4       workflowr_1.6.2   

loaded via a namespace (and not attached):
 [1] nlme_3.1-152       fs_1.5.0           webshot_0.5.2      httr_1.4.2        
 [5] rprojroot_2.0.2    tensorA_0.36.2     tools_4.1.1        backports_1.2.1   
 [9] bslib_0.3.1        utf8_1.2.2         R6_2.5.1           DBI_1.1.1         
[13] lazyeval_0.2.2     mgcv_1.8-36        colorspace_2.0-2   withr_2.4.2       
[17] tidyselect_1.1.1   compiler_4.1.1     git2r_0.28.0       rvest_1.0.1       
[21] xml2_1.3.2         labeling_0.4.2     sass_0.4.0         scales_1.1.1      
[25] checkmate_2.0.0    systemfonts_1.0.2  stringr_1.4.0      digest_0.6.28     
[29] yulab.utils_0.0.2  rmarkdown_2.11     svglite_2.0.0      pkgconfig_2.0.3   
[33] htmltools_0.5.2    fastmap_1.1.0      highr_0.9          htmlwidgets_1.5.4 
[37] rlang_0.4.11       rstudioapi_0.13    gridGraphics_0.5-1 jquerylib_0.1.4   
[41] generics_0.1.0     farver_2.1.0       jsonlite_1.7.2     magrittr_2.0.1    
[45] ggplotify_0.1.0    patchwork_1.1.1    Rcpp_1.0.7         munsell_0.5.0     
[49] fansi_0.5.0        lifecycle_1.0.1    stringi_1.7.5      whisker_0.4       
[53] yaml_2.2.1         mathjaxr_1.4-0     parallel_4.1.1     promises_1.2.0.1  
[57] crayon_1.4.1       lattice_0.20-44    splines_4.1.1      gridtext_0.1.4    
[61] pillar_1.6.3       cubature_2.0.4.2   markdown_1.1       corpcor_1.6.10    
[65] glue_1.4.2         evaluate_0.14      ggfun_0.0.4        vctrs_0.3.8       
[69] treeio_1.16.2      httpuv_1.6.3       gtable_0.3.0       purrr_0.3.4       
[73] tidyr_1.1.4        assertthat_0.2.1   xfun_0.26          tidytree_0.3.5    
[77] later_1.3.0        viridisLite_0.4.0  tibble_3.1.5       aplot_0.1.1       
[81] ellipsis_0.3.2    </code></pre>
</div>
</div>
</div>


<!-- Adjust MathJax settings so that all math formulae are shown using
TeX fonts only; see
http://docs.mathjax.org/en/latest/configuration.html.  This will make
the presentation more consistent at the cost of the webpage sometimes
taking slightly longer to load. Note that this only works because the
footer is added to webpages before the MathJax javascript. -->
<script type="text/x-mathjax-config">
  MathJax.Hub.Config({
    "HTML-CSS": { availableFonts: ["TeX"] }
  });
</script>




</div>
</div>

</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->
<script>
$(document).ready(function () {
  window.initializeCodeFolding("hide" === "show");
});
</script>

<script>
$(document).ready(function ()  {

    // temporarily add toc-ignore selector to headers for the consistency with Pandoc
    $('.unlisted.unnumbered').addClass('toc-ignore')

    // move toc-ignore selectors from section div to header
    $('div.section.toc-ignore')
        .removeClass('toc-ignore')
        .children('h1,h2,h3,h4,h5').addClass('toc-ignore');

    // establish options
    var options = {
      selectors: "h1,h2,h3",
      theme: "bootstrap3",
      context: '.toc-content',
      hashGenerator: function (text) {
        return text.replace(/[.\\/?&!#<>]/g, '').replace(/\s/g, '_');
      },
      ignoreSelector: ".toc-ignore",
      scrollTo: 0
    };
    options.showAndHide = true;
    options.smoothScroll = true;

    // tocify
    var toc = $("#TOC").tocify(options).data("toc-tocify");
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
